%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Stylish Article
% LaTeX Template
% Version 2.1 (1/10/15)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Mathias Legrand (legrand.mathias@gmail.com) 
% With extensive modifications by:
% Vel (vel@latextemplates.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------
\PassOptionsToPackage{table}{xcolor}
\PassOptionsToPackage{dvipsnames}{xcolor}

\documentclass[fleqn,10pt]{SelfArx} % Document font size and equations flushed left

\usepackage[english]{babel} % Specify a different language here - english by default

\usepackage{lipsum} % Required to insert dummy text. To be removed otherwise
\usepackage{tabularx}
\usepackage{multirow}
\setlength{\parindent}{0pt}

%----------------------------------------------------------------------------------------
%	COLUMNS
%----------------------------------------------------------------------------------------

\setlength{\columnsep}{0.55cm} % Distance between the two columns of text
\setlength{\fboxrule}{0.75pt} % Width of the border around the abstract

%----------------------------------------------------------------------------------------
%	COLORS
%----------------------------------------------------------------------------------------

\definecolor{color1}{RGB}{0,0,100} % Color of the article title and sections
\definecolor{color2}{RGB}{0,70,150} % Color of the boxes behind the abstract and headings

%----------------------------------------------------------------------------------------
%	HYPERLINKS
%----------------------------------------------------------------------------------------

\usepackage{hyperref} % Required for hyperlinks
\hypersetup{hidelinks,colorlinks,breaklinks=true,urlcolor=color2,citecolor=color1,linkcolor=color1,bookmarksopen=false,pdftitle={Title},pdfauthor={Author}}

%----------------------------------------------------------------------------------------
%	ARTICLE INFORMATION
%----------------------------------------------------------------------------------------

\JournalInfo{Laboratory of Data Mining Report} % Journal information
\Archive{} % Additional notes (e.g. copyright, DOI, review/research article)

\PaperTitle{Identification of putative drug targets for coronary artery disease} % Article title

\Authors{John Smith\textsuperscript{1}*, James Smith\textsuperscript{2}} % Authors
\affiliation{\textsuperscript{1}\textit{Department of Biology, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{\textsuperscript{2}\textit{Department of Chemistry, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{*\textbf{Corresponding author}: john@smith.com} % Corresponding author

\Keywords{Keyword1 --- Keyword2 --- Keyword3} % Keywords - if you don't want any simply remove all the text between the curly brackets
\newcommand{\keywordname}{Keywords} % Defines the keywords heading name

%----------------------------------------------------------------------------------------
%	ABSTRACT
%----------------------------------------------------------------------------------------

\Abstract{\lipsum[1]~}

%----------------------------------------------------------------------------------------

\begin{document}

\flushbottom % Makes all text pages the same height

\maketitle % Print the title and abstract box

\tableofcontents % Print the contents section

\thispagestyle{empty} % Removes page numbering from the first page

%----------------------------------------------------------------------------------------
%	ARTICLE CONTENTS
%----------------------------------------------------------------------------------------

\section*{Introduction} % The \section*{} command stops section numbering

\addcontentsline{toc}{section}{Introduction} % Adds this section to the table of contents
Drug repurposing is a strategy to identify new therapeutics uses for marketed drugs \cite{Polamreddy}. This procedure has known for time but only in recent years its use has increased due to new molecular discoveries, new high-throughput technologies and databases. 

The classic discovery of a new drug is a difficult and expensive process. Although investment in biomedical and pharmaceutical research and development has increased significantly in the past 20 years, the annual number of new treatments
approved by the US Food and Drug Administration (FDA) has not significantly increased \cite{Feixiong}. This could be due to a lack of well-establisehd  predictive pharmacokinetics/pharmacody-
namics approaches, and concerning safety and tolerability profiles
for new chemical entities from preclinical studies to clinical
trials\cite{Shih}. 

Discovering a new drug implies many resources and copious time, in average it takes 14 years to be able to have a product ready for the market, although all the efforts are often not payed off. 

Drug repurposing can be a good proposal to overcome some of the problems of classic drug discovery. For example it can drastically decrease the time because it is based on previous knowlege and drugs that have already been approved by the FDA, therefore also the costs can decrease because there is not the need of safety and efficacy evaluation.

Coronary artery disease (CAD), also knwon as coronary heart disease (CHD) or ischemic heart disease (IHD), is the most common type of heart disease. It is the leading cause of death in most of the western countries including Italy (World Health Organization data). The condition leads the formation of a waxy substance called plaque into coronary arteries decreasing the flow of oxygen-rich blood to the heart. The condition could worsen after the plaque rupture due to the formation of blood clot that could mostly (or completely) block blood flow through a coronary artery. The disease can weaken the heart muscle and lead to heart failure and arrhytmias, moreover it can cause angina, heart attack and sudden cardiac heart.
CAD can be prevent maintaining a healthy lifestyle with regular exercises and a balance diet, however when this is not enough the use of medication is needed. Since CAD is a complex disease caused by many factors, different kind of drugs that can be used, among which aspirin is the most knwon one. 
However in the worst cases, it is necessary to undergo surgical intervention such as coronary artery bypass surgery (CABG).

Since, as aforementioned, CAD is the leading cause of death in most of the countries, we think there is the unmet medical need to find new drugs able to cure the disease acting on diffent aspects genetics or not. Moreover having a large panel of medications to use could help the physicians to identify the right cure for the patients.

In this project therefore we apply NES$^2$RA\cite{NES2RA}, a pipeline that finds candidate genes to expand a known gene-network based PC-algorithm \cite{PC-alg}, with the aim to find marketed drug that can be repurposed for CAD.
Moreover we want to test if NES$^2$RA could be a useful tool for this process with the idea to propose it such as standard workflow to follow for drug repurposing. 

\section*{Materials and Methods}
\addcontentsline{toc}{section}{Materials and Methods} % Adds this section to the table of contents

\subsection*{Platforms and Datasources}
\addcontentsline{toc}{subsection}{Platforms and Datasources} % Adds this section to the table of contents

Nessra is designed in three steps in which first subsets data,
secondly runs the skeleton of the PC algorithm and finally aggregates results, computing the frequencies of the transcripts.

For the most computationally demanding step of Nessra, which consists in running the skeleton of the PC algorithm, the sigle gene expansions were run within the local gene@home \cite{boinc} BOINC (Berkeley Open Infrastructure for Network Computing) project, hosted by the TN-Grid platform. 
The platform allowed to distribute the expansions computational burden to committed volunteers' machines in order to reduce the running time and memory usage of the algorithm as much as possible.
This gene@home project was previously set up for the network expansion of genes whose expression data came from the FANTOM5 Database \cite{fantom}, so this guided the choice of the transcriptional data to run Nessra on.
To adress the drug repurposing task, Open Targets\cite{open} platform was chosen to select genes involved in the CAD disease.  


\subsection*{Pipeline}
\addcontentsline{toc}{subsection}{Pipeline} % Adds this section to the table of contents

Our pipeline can be summarized in  6 steps, that are briefly summarized below and are represented in Fig. \ref{Fig:pipe}.

\begin{itemize}[leftmargin=*]
	\item[1)] Starting from a list of 48 genes genetically associated with CAD, obtained form Open Targets, single gene expansions are preformed using NES$^2$RA.
	\item[2)] The resulting expansion lists are aggregated, obtaining a score for each isoform present in the expansion lists. This score is meant to recapitulate the overall interaction of an isoform with the list of CAD genes. Isoforms are then filtered according to the genetic association scores of the CAD genes they interact with.
	\item[3)] The resulting list of isoforms is converted in a list of genes. Among these genes, we select those that are targets of clinically approved drugs. The result of this step is a list of preliminary putative drug repurposing targets.
	\item[4)] The list of preliminary targets is further analyzed, performing single gene expansions using NES$^2$RA.
	\item[5)] The Weighted Jaccard Similarity (WJS) is used to compare a ranked list of genes genetically associated with CAD, obtained form Open Targets, with the ranked lists of isoforms, obtained using NES$^2$RA.
	\item[6)] A statistical analysis of the WJS scores is employed in order to identify which genes present a significant interaction with the list of CAD genes. The result of this step is a final list of putative drug repurposing targets.
\end{itemize}

The rationale of steps 4), 5) and 6) is the identification of genes, among the preliminary targets, that present a strong interaction with CAD associated genes. This analysis allows us to have a larger confidence on the final list of putative targets, since their interactions with CAD genes has been tested twice (they were found in the expansions of CAD genes and CAD genes are present in their expansion).

Each step of the pipeline is described in further details below.
 
\cite{OpenTargetPlatform}
\cite{fantom}
\cite{realBoinc}
\cite{WGS}

\begin{figure}
	\includegraphics[width=1.1\linewidth, trim = 2cm 0 1.5cm 0.4cm, clip = true]{pipeline.png}
	\caption{Pipeline sketch. Color legend = ...}
	\label{Fig:pipe}
\end{figure}

\subsubsection*{Selection of CAD genes}
Among the features of Open Target, there's the search for genes somehow associated to a specific illness. 
We downloaded from the website the list of 643 genetically associated CAD genes and sorted them based on the overall association score and focused on the first 100 of them. 
The FANTOM5 database is built on gene isoforms and we were provided with an annotation file to perform the mapping of a gene into its corresponding isoforms.
Some of the downloaded CAD genes were not present in the annotation, therefore we had to exclude them aprioristically.
Given unlimited time, we would have expanded all the selected 100 genes, yet we had to restrain our search to 41 of them, aside from the 5 that had already been expanded by previous runs of the gene@home BOINC project.

\subsubsection*{NESSRA expansion of CAD associated genes}
The 41 cad genetically associated genes, totalling to about 180 isoforms, were turned into one gene network files as inputs for NESSRA.
The algorithm outputs a series of expansion networks, each of which is a list of nodes ranked on the reported score, which is a measure of the strenght of the supposed correlation between that node and the input one. 

\subsubsection*{Score aggregation of each isoform}
The output lists were merged on the isoforms,summing their scores in order to rank them based on how many times each isoform appeared in the various expansions lists.

\subsubsection*{Filtering based on association score}
The output lists were shortened using as weight the genetic association of the gene that corresponds to the input isoform, going from a maximum of 100 nodes (in the case of highest genetic association) to a minimum of 1 at steps of 1. 

The resulting n isoforms were ranked according to the aggregated score described previously. Then, the ranked list of isoforms was converted in a ranked list of genes. The rank of a gene was obtained as the minimum of the ranks of its isoforms. 

\subsubsection*{Selection of target genes}

The Open Tragets pyhton API [reference?] was used to query which of these genes are known to be targets of clinically approved drugs. We obtained a list of 148 target genes, which are the only ones that are considered in the following analyses.

\subsubsection*{NES$^{\textbf{2}}$RA expansion of target genes}

In order to identify a subset of genes with a strong association with CAD, we selected the isoforms corresponding to these 148 target genes and we performed single isoform expansions using NES$^2$RA. Only the isoforms that passed the previous filter were considered. We expect NES$^2$RA expansions of genes that have strong interaction with CAD genes to have a large overlap with the list of CAD genes we started form.

\subsubsection*{Comparison of expanded lists to CAD genes}

In order to quantify the overlap between the ranked list of CAD genes and the ranked expansion lists previously obtained, we used the weighted Jaccard similarity (WJS), which we define below.\medskip

\noindent
\textit{Definition: given two weighted list of items, $\rho$ and $\sigma$, of equal length $N$, their weighted Jaccard similarity, $WJS(\rho, \sigma)$, is defined as:}

$$
WJS(\rho, \sigma) = \dfrac{\sum_{i=1}^Nmin(\rho_i,\sigma_i)}{\sum_{i=1}^Nmax(\rho_i,\sigma_i)}
$$

\noindent
\textit{where $\rho_i$ and $\sigma_i$ are the weights corresponding to the same feature $i$.}\medskip

In our analysis the weight of a feature $i$ (gene or isoform) in a ranked list $\rho$ is computed as $length(\rho) - rank_{\rho}(i) + 1$. This allows to assign large weights to high ranking features.

Since in principle the lists of features that we want to compare with the WJS do not contain the same elements, we need to add the missing elements to those lists. Given two lists of features $\rho$ and $\sigma$, containing different elements, we added the missing elements of one list to the other and viceversa. The missing elements are added as ties in the last position of the ranking.

In order to compute the WJS of the ranked list of CAD genes and a ranked list of isoforms obtained using NES$^2$RA, we have to either convert the ranked list of CAD genes in a ranked list of isoforms or convert the ranked list of isoforms obtained using NES$^2$RA in a ranked list of genes. We explored both options:

\begin{itemize}[leftmargin=*]
	\item \textit{Convert NES$^2$RA isoforms into genes}: Given list of isoforms obtained with NES$^2$RA, we can convert it in a list of genes, assigning to each gene the largest relative frequency of its isoforms. Then the genes were ranked according to relative frequency.
	\item \textit{Convert CAD genes into isoforms}: The ranked list of CAD genes was converted to a weighted list of isoforms, were the weight of each isoform corresponded to the rank of the genes. Then the list of isoforms was ranked according to the weights. This way, isoforms corresponding to the same gene presented the same rank.
\end{itemize}

We computed the WJS of the ranked list of CAD genes (respectively, isoforms) and the ranked lists of genes (respectively, isoforms) obtained with NES$^2$RA.

\subsubsection*{Significance analysis} 

The scores obtained in different comparisons are not directly comparable between them, since they depend on the length of the lists. In order to obtain values that can be compared directly, we used a permutation approach to estimate the distribution of scores of lists of different length. For each length present, we generated 2000 random lists of genes or isoforms, we computed the WJS and we generated a distribution of the scores. Then, we used this distributions to compute p-values.

Using a significance level of 0.05 (?), we obtained n genes and m isoforms with expansion lists enriched in CAD gene. The list of isoforms was also converted in a list of genes.

\subsection*{Enrichment analysis}
\addcontentsline{toc}{subsection}{Enrichment analysis} % Adds this section to the table of contents

In order to gain insight on the biological role of the identified genes, we performed functional enrichment analysis. We used both Open Targets to perform enrichment analysis against the Biological Processes form Gene Ontology.

2229 genes, targets of approved drugs, ToppGenes, BP

149 genes, ToppGenes, BP

36 genes, BP Open Targets

18 genes, BP Open Targets

26 genes, diseases, ToppGenes


\section*{Results}
\addcontentsline{toc}{section}{Results} % Adds this section to the table of contents
Matteo

The list of 149 target genes is reported in the Supplementary Material (Tab. \ref{tab:194genes}).

The list of 36 significant genes are reported in Tab. \ref{tab:sign}. In this table are also indicated the genes found by option2.

\renewcommand{\arraystretch}{1.1}

\newcolumntype{R}{>{\raggedleft\arraybackslash}X}
\begin{table}[htb]
	\centering
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\linewidth}{XRRRR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Gene} & \textbf{\color{white} Number of isoforms} & \textbf{\color{white} Minimum p-value} & \textbf{\color{white} Number of isoforms} & \textbf{\color{white} Minimum p-value} \\ 
		CASP7 & 1 & 0 & 1 & 0 \\ 
		COL1A1 & 1 & 0 & 1 & 0 \\ 
		COL4A1 & 4 & 0 & 4 & 0 \\ 
		COL4A2 & 6 & 0 & 3 & 0 \\ 
		CTGF & 2 & 0 & 1 & 0 \\ 
		FN1 & 8 & 0 & 8 & 0 \\ 
		IL1R1 & 1 & 0 & 1 & 0 \\ 
		INSR & 1 & 0 & 1 & 4.0e-02 \\ 
		JAK1 & 1 & 0 &  &  \\ 
		NDUFA4L2 & 1 & 0 & 1 & 0 \\ 
		PDE4B & 2 & 0 & 1 & 1.7e-02 \\ 
		PDE4D & 3 & 0 & 2 & 7.5e-03 \\ 
		RARA & 1 & 0 &  &  \\ 
		TLR4 & 2 & 0 &  &  \\ 
		PIK3CG & 1 & 1.2e-02 &  &  \\ 
		FNTB & 1 & 1.6e-02 &  &  \\ 
		CSF1 & 1 & 1.7e-02 &  &  \\ 
		TSG101 & 1 & 1.7e-02 &  &  \\ 
		F5 & 1 & 2.3e-02 &  &  \\ 
		CDK14 & 1 & 2.6e-02 & 1 & 2.2e-02 \\ 
		TGFB1 & 2 & 2.7e-02 &  &  \\ 
		ANPEP & 1 & 2.9e-03 &  &  \\ 
		AVPR1A & 1 & 2.9e-03 & 1 & 4.1e-03 \\ 
		DNMT3A & 1 & 2.9e-03 &  &  \\ 
		PIK3CA & 1 & 2.9e-03 & 1 & 3.9e-02 \\ 
		TNFRSF1A & 1 & 2.9e-03 &  &  \\ 
		IL6 & 1 & 3.2e-02 & 1 & 1.7e-02 \\ 
		FLT1 & 1 & 3.5e-02 &  &  \\ 
		SEBOX & 1 & 3.8e-02 &  &  \\ 
		COL5A3 & 1 & 4.4e-02 &  &  \\ 
		GAA & 1 & 4.4e-02 &  &  \\ 
		KCNJ2 & 1 & 4.6e-02 & 1 & 1.1e-02 \\ 
		FGA & 1 & 4.9e-02 &  &  \\ 
		NAMPT & 1 & 5.3e-03 &  &  \\ 
		IFNAR2 & 1 & 7.5e-03 &  &  \\ 
		PTGER4 & 1 & 7.5e-03 &  &  \\ 
	\end{tabularx}
	\smallskip
	\caption{resulting genes}
	\label{tab:sign}
\end{table}

The results of the enrichment analysis against the Biological Processes form Gene Ontology are reported in Tab. \ref{tab:GO36_1} and Tab. \ref{tab:GO36_2} for the results of option1 and in Tab. \ref*{tab:GO18} for the results of option2.

The drugs that target the genes resulting from option1 are reported in Tab. \ref{tab:drugs}.

19 genes found were already associated with CAD. 4 of them are also target of drugs used to treat CAD. The list of genes and their association levels are reported in Fig. \ref{fig:19CAD}.


\begin{figure}[ht]
	\includegraphics[width=\linewidth]{already_in}
	\smallskip
	\caption{Putative targets already associated with CAD. Four of them are also already targeted in CAD.}
	\label{fig:19CAD}
\end{figure}

results of the enrichment analysis

\section*{Discussion}
\addcontentsline{toc}{section}{Discussion} % Adds this section to the table of contents
Because NESS2RA gene -isoforms- expansions are meant to explore through a local gene network approach the space of putative causal relationships between genes, without any further biological heuristics,NESS2RA outcome results conservative? enough to the further application of Filtering, Aggregation, Ranking and Comparison methods.



Performing Enrichment analysis on Open Targets of the 36 genes resulted positive to the Significance analysis, GO terms (see Table 5) such as platelet activation, Tyrosine kinase signaling pathway,regulation of blood circulation, cell chemotaxis, angiogenesis and striated muscle contraction are present. 

Performing Enrichment analysis on Open Targets of the 18 genes positive to the Significance analysis, GO terms (see Table 5) such as circulatory system development, cardiac muscle contraction and relaxation (pvalue $7.43^04$), blood circulation, leukocytes migration are present (among others more general terms such as responce to compounds?) 
 
Performing it on Top genes, specifically looking at Disease terms - pvalue

gene that are already targets of drugs used for cad

interpretation of the enrichment analysis

discussion of new targets and drugs

\section*{Conclusions}
\addcontentsline{toc}{section}{Conclusions} % Adds this section to the table of contents

The pipeline described here was intended to investigate on the potentialities of NESS2RA implementation in respect to a drug repurposing objective.
NESS2RA allows to search for putative causal relationships between genes (specifically isoforms) but then a delicate phase of filtering, aggregation and validation has to follow in order to identify most promising gene relations.

necessity of experimental validations

\phantomsection
%----------------------------------------------------------------------------------------
%	REFERENCE LIST
%----------------------------------------------------------------------------------------
\phantomsection
\bibliographystyle{unsrt}
%\bibliography{biblio}
\begin{thebibliography}{1}
	
	\bibitem{Polamreddy} Polamreddy P, Gattu N. The drug repurposing landscape from 2012 to 2017: evolution, challenges, and possible solutions. Drug Discov Today. 2018 Dec 1.	
	
	\bibitem{Feixiong} Cheng F, Desai RJ, Handy DE, Wang R, Schneeweiss S, Barabási AL, Loscalzo J. Network-based approach to prediction and population-based validation of in silico drug repurposing. Nat Commun. (2018).
	
	\bibitem{Shih} Shih, H. P., Zhang,X. and Aronov, A. M. Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications. Nat. Rev. Drug Discov.17, 19–33 (2017).

	\bibitem{NES2RA} Asnicar F, Masera L, NES2RA: network expansion by stratified variable subsetting and ranking aggregation. INTERNATIONAL JOURNAL OF HIGH PERFORMANCE COMPUTING APPLICATIONS (2018)
	
	\bibitem{PC-alg}Spirtes P and Glymour C An algorithm for fast recovery
of sparse causal graphs. Social Science Computer, Review, (1991).  

	\bibitem{OpenTargetPlatform}Denise Carvalho-Silva, Open Targets Platform: new developments and updates two years on, Nucleic Acids Research 
	
	\bibitem{realBoinc}Anderson DP, BOINC: A system for public-resource computing and storage. In: Proceedings of the 5th IEEE/ ACM international workshop on grid computing, GRID ‘04,Washington, DC, US (2004)
	
	\bibitem{WGS}Improved Consistent Sampling, Weighted Minhash and L1 Sketching, Sergey Ioffe, 2010, ICDM
	
	\bibitem{boinc} Asnicar, F. et al. NES2RA. Int. J. High Perform. Comput. Appl. 32, 380–392 (2016).
	
	
	\bibitem{fantom} Noguchi, S. et al. FANTOM5 CAGE profiles of human and mouse samples. Sci Data 4, 170112 (2017).
	
	
	\bibitem{open} Koscielny, G. et al. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 45, D985–D994 (2017).
	
	
\end{thebibliography}

%----------------------------------------------------------------------------------------
%	SUPPLEMENTARY MATERIAL
%----------------------------------------------------------------------------------------

\pagebreak
\onecolumn

\section*{SUPPLEMENTARY MATERIAL}

\addcontentsline{toc}{section}{Supplementary Material} % Adds this section to the table of contents


\begin{table}[ht]
	\centering
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\textwidth}{XXXXXX}
		NDUFA4L2 & JAK1 & CDK16 & SRD5A1 & PRKCD & FLT4 \\ 
		FN1 & KCNJ2 & PIM2 & FGFR4 & ABCA1 & DHODH \\ 
		COL4A1 & FDPS & COL27A1 & TUBA1B & ERBB3 & FNTB \\ 
		COL4A2 & CPT2 & ALOX5 & PIK3CG & NR3C1 & TNFSF12 \\ 
		CTGF & LAMB3 & CD70 & GABBR1 & HGF & TGFB3 \\ 
		F5 & PCSK9 & GSK3B & PDE2A & BIRC2 & KCNH8 \\ 
		SEBOX & FLT1 & SLC29A1 & MMP9 & ERAP1 & EGLN2 \\ 
		TGFB1 & VEGFB & PIM3 & PSCA & OPRL1 & METAP2 \\ 
		F2 & POLD1 & CSF1 & CALM1 & GSTP1 & LAMA2 \\ 
		TLR4 & IL1R1 & CASP7 & PDE8B & PRKACA & NDUFA13 \\ 
		COL5A3 & XPNPEP2 & GAA & DNMT3A & RAMP2 & MAPKAPK5 \\ 
		COL1A1 & NR1H3 & PDE4B & IL6 & HSP90AA1 & CD276 \\ 
		FGA & NFE2L2 & PTGS2 & EPHA2 & INSR & HDAC5 \\ 
		FOLR1 & GBA & NCSTN & PIK3CA & MAP3K9 & DRD2 \\ 
		DHCR24 & PLA2G2A & NDUFS2 & TNFRSF8 & TNFSF13 & NTRK1 \\ 
		GLP2R & ACAT1 & TSG101 & LTB4R & CASP8 & PIK3R3 \\ 
		TNFRSF1A & FDFT1 & PLK1 & VDR & CACNG4 & MALT1 \\ 
		AVPR1A & HLA-DRB1 & NDUFB7 & EEF2 & NFE2 & SLC6A9 \\ 
		FGG & IFNGR2 & BCL2L1 & SCN9A & LTBR & HDAC6 \\ 
		NAMPTL & PLA2G7 & IDH2 & PTH1R & EPHB6 & PTGER4 \\ 
		SRD5A3 & CHEK2 & PSMA4 & P4HB & GART & KCNK10 \\ 
		PDE4D & ATP1B3 & ENG & CFD & KCNH2 & CD44 \\ 
		HAMP & ANPEP & EPHA1 & ARAF & S1PR5 & P4HTM \\ 
		RARA & COL6A3 & TNFRSF10B & SIGMAR1 & TUBB & MET \\ 
		NAMPT & COL11A2 & CDK14 & IFNAR2 & KCNC3 &  \\ 
	\end{tabularx}
	\smallskip
	\caption{List of tagret genes}
	\label{tab:194genes}
\end{table}

\renewcommand{\arraystretch}{1.2}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lXlll}
		\rowcolor{NavyBlue!80}
		 \textbf{\color{white} DRUG} & \textbf{\color{white} TARGET} & \textbf{\color{white} DISEASE} & \textbf{\color{white} PAHSE} & \textbf{\color{white} MOLECULE TYPE} \\
		SORAFENIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		COLLAGENASE CLOSTRIDIUM HISTOLYTICUM & COL5A3 & Dupuytren Contracture & Phase IV & Enzyme \\ 
		APREMILAST & PDE4D & immune system disease & Phase IV & Small molecule \\ 
		INSULIN GLARGINE & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		METFORMIN & NDUFA4L2 & type II diabetes mellitus & Phase IV & Small molecule \\ 
		INSULIN HUMAN & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		ADAPALENE & RARA & acne & Phase IV & Small molecule \\ 
		THEOPHYLLINE & PDE4B & Bronchiectasis & Phase IV & Small molecule \\ 
		LENVATINIB & FLT1 & thyroid carcinoma & Phase IV & Small molecule \\ 
		INSULIN PORK & INSR & diabetes mellitus & Phase IV & Protein \\ 
		PEGINTERFERON ALFA-2A & IFNAR2 & hepatitis C infection & Phase IV & Protein \\ 
		INTERFERON BETA-1A & IFNAR2 & multiple sclerosis & Phase IV & Protein \\ 
		FLAVOXATE & PDE4D & pain & Phase IV & Small molecule \\ 
		DECITABINE & DNMT3A & chronic myelomonocytic leukemia & Phase IV & Small molecule \\ 
		AZACITIDINE & DNMT3A & myelodysplastic syndrome & Phase IV & Small molecule \\ 
		INSULIN LISPRO & INSR & type I diabetes mellitus & Phase IV & Protein \\ 
		ACITRETIN & RARA & squamous cell carcinoma & Phase IV & Small molecule \\ 
		TRETINOIN & RARA & acne & Phase IV & Small molecule \\ 
		PENTOXIFYLLINE & PDE4B & chronic kidney disease & Phase IV & Small molecule \\ 
		PEGINTERFERON ALFA-2B & IFNAR2 & Chronic Hepatitis C infection & Phase IV & Protein \\ 
		INSULIN DETEMIR & INSR & diabetes mellitus & Phase IV & Protein \\ 
		DRONEDARONE & KCNJ2 & atrial fibrillation & Phase IV & Small molecule \\ 
		ISOTRETINOIN & RARA & seborrheic dermatitis & Phase IV & Small molecule \\ 
		DIPYRIDAMOLE & PDE4D & coronary artery disease & Phase IV & Small molecule \\ 
		INSULIN ASPART & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		CRISABOROLE & PDE4B & seborrheic dermatitis & Phase IV & Small molecule \\ 
		SUNITINIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		INSULIN GLULISINE & INSR & type I diabetes mellitus & Phase IV & Protein \\ 
		INSULIN SUSP ISOPHANE BEEF & INSR & type I diabetes mellitus & Phase IV & Protein \\ 
		NINTEDANIB & FLT1 & idiopathic pulmonary fibrosis & Phase IV & Small molecule \\ 
		ROFLUMILAST & PDE4B & chronic obstructive pulmonary disease & Phase IV & Small molecule \\ 
		INTERFERON BETA-1B & IFNAR2 & relapsing-remitting multiple sclerosis & Phase IV & Protein \\ 
		CONIVAPTAN & AVPR1A & heart failure & Phase IV & Small molecule \\ 
		TAZAROTENE & RARA & acne & Phase IV & Small molecule \\ 
		TOFACITINIB & JAK1 & rheumatoid arthritis & Phase IV & Small molecule \\ 
		ETRETINATE & RARA & psoriasis & Phase IV & Small molecule \\ 
		INSULIN DEGLUDEC & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		PAZOPANIB & FLT1 & renal cell carcinoma & Phase IV & Small molecule \\ 
		INSULIN ASPART PROTAMINE RECOMBINANT & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		SILTUXIMAB & IL6 & Giant Lymph Node Hyperplasia & Phase IV & Antibody \\ 
		PEGINTERFERON BETA-1A & IFNAR2 & multiple sclerosis & Phase IV & Protein \\ 
		INSULIN PURIFIED PORK & INSR & diabetes mellitus & Phase IV & Protein \\ 
		ANAKINRA & IL1R1 & immune system disease & Phase IV & Protein \\ 
		INTERFERON ALFA-2B & IFNAR2 & Chronic Hepatitis C infection & Phase IV & Protein \\ 
		FIBRINOLYSIN, HUMAN & FGA & Recurrent thrombophlebitis & Phase IV & Unknown \\ 
		VASOPRESSIN & AVPR1A & sudden cardiac arrest & Phase IV & Protein \\ 
		DROTRECOGIN ALFA (ACTIVATED) & F5 & sepsis & Phase IV & Unknown \\ 
		INTERFERON ALFACON-1 & IFNAR2 & hepatitis C infection & Phase IV & Protein \\ 
		INSULIN SUSP ISOPHANE RECOMBINANT HUMAN & INSR & diabetic ketoacidosis & Phase IV & Protein \\ 
		MIGLITOL & GAA & diabetes mellitus & Phase IV & Small molecule \\ 
		INSULIN SUSP ISOPHANE SEMISYNTHETIC PURIFIED HUMAN & INSR & diabetes mellitus & Phase IV & Protein \\ 
		TIVOZANIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		AXITINIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		REGORAFENIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		DESMOPRESSIN & AVPR1A & hemorrhage & Phase IV & Protein \\ 
		RUXOLITINIB & JAK1 & polycythemia vera & Phase IV & Small molecule \\ 
		INSULIN PURIFIED BEEF & INSR & diabetes mellitus & Phase IV & Protein \\ 
		VANDETANIB & FLT1 & thyroid carcinoma & Phase IV & Small molecule \\ 
		ALITRETINOIN & RARA & neoplasm & Phase IV & Small molecule \\ 
		INSULIN LISPRO PROTAMINE RECOMBINANT & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		DYPHYLLINE & PDE4B & obstructive lung disease & Phase IV & Small molecule \\ 
		BARICITINIB & JAK1 & immune system disease & Phase IV & Small molecule \\ 
		AMLEXANOX & PDE4D & obstructive lung disease & Phase IV & Small molecule \\ 
		OCRIPLASMIN & FN1 & macular holes & Phase IV & Enzyme \\ 
		DINOPROSTONE & PTGER4 & pregnancy & Phase IV & Small molecule \\ 
		INTERFERON ALFA-2A & IFNAR2 & lymphoma & Phase IV & Protein \\ 
		COPANLISIB & PIK3CG & non-Hodgkins lymphoma & Phase IV & Small molecule \\ 
		INSULIN ZINC SUSP RECOMBINANT HUMAN & INSR & diabetes mellitus & Phase IV & Protein \\ 
		INTERFERON ALFA-N3 & IFNAR2 & severe acute respiratory syndrome & Phase IV & Protein \\ 
	\end{tabularx}
	\smallskip
	\caption{List of drugs targeting the 36 genes. Only Phase IV drugs are reportes.}
	\label{tab:drugs}
\end{table}


\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		response to organic substance & 1.7e-09 & positive regulation of response to external stimulus & 2.1e-03 \\ 
		cellular response to organic ubstance & 3.0e-09 & movement of cell or subcellular component & 2.4e-03 \\ 
		leukocyte migration & 3.8e-08 & intracellular signal transduction & 2.6e-03 \\ 
		circulatory system development & 5.3e-08 & regulation of transport & 2.6e-03 \\ 
		cellular response to chemical stimulus & 1.4e-07 & cellular response to environmental stimulus & 2.7e-03 \\ 
		regulation of multicellular organismal process & 4.6e-07 & cellular response to abiotic stimulus & 2.7e-03 \\ 
		response to chemical & 7.0e-07 & cell communication & 2.7e-03 \\ 
		blood circulation & 1.8e-06 & regulation of phosphorylation & 2.7e-03 \\ 
		multicellular organismal process & 2.0e-06 & cellular response to external stimulus & 2.8e-03 \\ 
		circulatory system process & 2.2e-06 & regulation of cytokine production & 2.9e-03 \\ 
		myeloid leukocyte migration & 2.5e-06 & cellular response to biotic stimulus & 2.9e-03 \\ 
		cellular response to oxygen-containing compound & 3.2e-06 & interferon-gamma production & 3.0e-03 \\ 
		cellular response to organonitrogen compound & 4.2e-06 & system process & 3.1e-03 \\ 
		response to external stimulus & 4.4e-06 & cellular developmental process & 3.2e-03 \\ 
		cellular response to endogenous stimulus & 8.3e-06 & response to bacterium & 3.4e-03 \\ 
		response to endogenous stimulus & 8.4e-06 & cell proliferation & 3.4e-03 \\ 
		response to organonitrogen compound & 1.1e-05 & cellular response to cytokine stimulus & 3.6e-03 \\ 
		immune response & 1.1e-05 & regulation of cell adhesion & 3.7e-03 \\ 
		cellular response to mechanical stimulus & 1.3e-05 & cellular response to acid chemical & 3.8e-03 \\ 
		protein phosphorylation & 1.6e-05 & leukocyte chemotaxis & 3.8e-03 \\ 
		cellular response to nitrogen compound & 2.3e-05 & collagen-activated signaling pathway & 3.9e-03 \\ 
		cell surface receptor signaling pathway & 3.3e-05 & secretion & 4.2e-03 \\ 
		regulation of response to cytokine stimulus & 3.6e-05 & response to mechanical stimulus & 5.0e-03 \\ 
		response to nitrogen compound & 4.7e-05 & regulation of heart contraction & 5.3e-03 \\ 
		extracellular structure organization & 4.9e-05 & positive regulation of biological process & 5.5e-03 \\ 
		response to stimulus & 5.4e-05 & blood coagulation & 5.6e-03 \\ 
		immune system process & 5.5e-05 & positive regulation of cell migration & 5.6e-03 \\ 
		response to oxygen-containing compound & 6.7e-05 & regulation of secretion by cell & 6.1e-03 \\ 
		multi-organism process & 6.9e-05 & hemostasis & 6.1e-03 \\ 
		regulation of cell communication & 7.0e-05 & positive regulation of nitric-oxide synthase biosynthetic process & 6.2e-03 \\ 
		tube morphogenesis & 7.1e-05 & coagulation & 6.4e-03 \\ 
		regulation of signaling & 8.4e-05 & positive regulation of cell motility & 7.2e-03 \\ 
		wound healing & 8.6e-05 & cell activation & 7.4e-03 \\ 
		response to cytokine & 9.8e-05 & defense response & 7.8e-03 \\ 
		blood vessel development & 1.1e-04 & positive regulation of inflammatory response & 7.9e-03 \\ 
		response to organic cyclic compound & 1.3e-04 & negative regulation of relaxation of cardiac muscle & 8.2e-03 \\ 
		positive regulation of multicellular organismal process & 1.4e-04 & positive regulation of secretion by cell & 8.3e-03 \\ 
		cellular response to stimulus & 1.5e-04 & positive regulation of cellular component movement & 9.0e-03 \\ 
		regulation of immune response & 1.7e-04 & localization & 9.1e-03 \\ 
		anatomical structure development & 1.7e-04 & relaxation of cardiac muscle & 9.3e-03 \\ 
		vasculature development & 1.8e-04 & response to other organism & 9.9e-03 \\ 
		response to lipopolysaccharide & 1.9e-04 & response to external biotic stimulus & 1.0e-02 \\ 
		cardiovascular system development & 2.0e-04 & regulation of phosphate metabolic process & 1.1e-02 \\ 
		granulocyte migration & 2.0e-04 & regulation of phosphorus metabolic process & 1.1e-02 \\ 
		negative regulation of multicellular organismal process & 2.1e-04 & positive regulation of cell adhesion & 1.1e-02 \\ 
		regulation of cellular component movement & 2.3e-04 & regulation of secretion & 1.2e-02 \\ 
		cell migration & 2.4e-04 & regulation of defense response & 1.2e-02 \\ 
		extracellular matrix organization & 2.7e-04 & regulation of system process & 1.2e-02 \\ 
		phosphorylation & 2.7e-04 & peptidyl-tyrosine phosphorylation & 1.3e-02 \\ 
		regulation of response to stimulus & 2.8e-04 & positive regulation of interferon-gamma production & 1.3e-02 \\ 
		response to molecule of bacterial origin & 2.9e-04 & negative regulation of cytokine production & 1.3e-02 \\ 
		regulation of localization & 3.2e-04 & regulation of MAPK cascade & 1.3e-02 \\ 
		regulation of immune system process & 3.2e-04 & nitric-oxide synthase biosynthetic process & 1.3e-02 \\ 
		cardiac muscle contraction & 3.4e-04 & regulation of nitric-oxide synthase biosynthetic process & 1.3e-02 \\ 
		signaling & 3.8e-04 & peptidyl-tyrosine modification & 1.3e-02 \\ 
		signal transduction & 4.2e-04 & positive regulation of protein phosphorylation & 1.3e-02 \\ 
		blood vessel morphogenesis & 4.4e-04 & positive regulation of secretion & 1.4e-02 \\ 
		response to wounding & 4.6e-04 & cytokine-mediated signaling pathway & 1.4e-02 \\ 
		tube development & 5.2e-04 & immune effector process & 1.5e-02 \\ 
		locomotion & 5.5e-04 & positive regulation of immune response & 1.5e-02 \\ 
		heart contraction & 6.0e-04 & response to biotic stimulus & 1.5e-02 \\ 
		regulation of signal transduction & 6.4e-04 & regulation of locomotion & 1.6e-02 \\ 
		cytokine production & 6.5e-04 & cellular response to lipid & 1.6e-02 \\ 
		MAPK cascade & 6.8e-04 & macromolecule modification & 1.8e-02 \\ 
		system development & 7.3e-04 & chemotaxis & 1.8e-02 \\
		heart process & 7.4e-04 & taxis & 1.8e-02 \\	
		localization of cell & 7.5e-04 & regulation of cytokine production involved in immune response & 1.8e-02 \\ 
	\end{tabularx}
	\smallskip
	\caption{GO terms enriched in the list of 36 (continued)}
	\label{tab:GO36_1}
\end{table}
			
\begin{table}[!htb]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		cell motility & 7.5e-04 & regulation of protein modification process & 1.9e-02 \\ 
		regulation of cytokine-mediated signaling pathway & 7.5e-04 & positive regulation of phosphorylation & 2.0e-02 \\ 
		platelet activation & 7.8e-04 & enzyme linked receptor protein signaling pathway & 2.4e-02 \\ 
		transmembrane receptor protein tyrosine kinase signaling pathway & 8.1e-04 & regulation of response to external stimulus & 2.4e-02 \\ 
		developmental process & 8.2e-04 & positive regulation of intracellular signal transduction & 2.4e-02 \\ 
		signal transduction by protein phosphorylation & 8.3e-04 & positive regulation of response to macrophage colony-stimulating factor & 2.5e-02 \\ 
		response to lipid & 8.4e-04 & positive regulation of cellular response to macrophage colony-stimulating factor stimulus & 2.5e-02 \\ 
		positive regulation of locomotion & 9.3e-04 & positive regulation of signal transduction & 2.6e-02 \\ 
		regulation of blood circulation & 9.6e-04 & positive regulation of immune system process & 2.8e-02 \\ 
		anatomical structure formation involved in morphogenesis & 9.8e-04 & regulation of intracellular signal transduction & 2.9e-02 \\ 
		phosphate-containing compound metabolic process & 1.0e-03 & mononuclear cell migration & 3.3e-02 \\ 
		phosphorus metabolic process & 1.1e-03 & regulation of cardiac muscle contraction & 3.3e-02 \\ 
		multicellular organism development & 1.1e-03 & lymphocyte activation involved in immune response & 3.4e-02 \\ 
		cell chemotaxis & 1.1e-03 & positive regulation of phosphorus metabolic process & 3.5e-02 \\ 
		positive regulation of cell proliferation & 1.1e-03 & positive regulation of phosphate metabolic process & 3.5e-02 \\ 
		cellular response to lipopolysaccharide & 1.1e-03 & regulation of biological quality & 3.5e-02 \\ 
		positive regulation of cytokine production & 1.2e-03 & tissue development & 3.9e-02 \\ 
		regulation of protein phosphorylation & 1.3e-03 & regulation of cell migration & 4.0e-02 \\ 
		anatomical structure morphogenesis & 1.4e-03 & protein modification process & 4.1e-02 \\ 
		angiogenesis & 1.5e-03 & cellular protein modification process & 4.1e-02 \\ 
		positive regulation of response to stimulus & 1.5e-03 & transport & 4.2e-02 \\ 
		cell differentiation & 1.5e-03 & muscle system process & 4.2e-02 \\ 
		cellular response to molecule of bacterial origin & 1.5e-03 & positive regulation of production of molecular mediator of immune response & 4.2e-02 \\ 
		secretion by cell & 1.5e-03 & regulation of leukocyte migration & 4.2e-02 \\ 
		regulation of cell proliferation & 1.6e-03 & positive regulation of metabolic process & 4.6e-02 \\ 
		collagen-activated tyrosine kinase receptor signaling pathway & 1.7e-03 & cytokine production involved in immune response & 4.6e-02 \\ 
		regulation of interferon-gamma production & 1.7e-03 & positive regulation of developmental process & 4.8e-02 \\ 
		striated muscle contraction & 1.9e-03 & positive regulation of immune effector process & 4.8e-02 \\ 
		inflammatory response & 2.0e-03 &  &  \\
	\end{tabularx}
	\smallskip
	\caption{GO terms enriched in the list of 36.}
	\label{tab:GO36_2}
\end{table}

\begin{table}[!htb]
		\centering
		\scriptsize
		\rowcolors{2}{NavyBlue!10}{white}
		\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		cellular response to organic substance & 5.1e-07 & response to interleukin-6 & 9.1e-03 \\ 
		cellular response to chemical stimulus & 5.8e-06 & signaling & 9.2e-03 \\ 
		response to organic substance & 7.6e-06 & cardiac muscle contraction & 9.5e-03 \\ 
		response to organonitrogen compound & 4.9e-05 & regulation of system process & 9.6e-03 \\ 
		regulation of multicellular organismal process & 5.7e-05 & response to external stimulus & 1.0e-02 \\ 
		cellular response to oxygen-containing compound & 1.2e-04 & regulation of transport & 1.1e-02 \\ 
		leukocyte migration & 1.3e-04 & regulation of cardiac muscle cell contraction & 1.2e-02 \\ 
		collagen-activated tyrosine kinase receptor signaling pathway & 1.3e-04 & extracellular matrix organization & 1.4e-02 \\ 
		response to nitrogen compound & 1.4e-04 & response to organic cyclic compound & 1.6e-02 \\ 
		circulatory system development & 2.2e-04 & regulation of actin filament-based movement & 1.6e-02 \\ 
		response to endogenous stimulus & 2.8e-04 & platelet activation & 1.6e-02 \\ 
		collagen-activated signaling pathway & 3.1e-04 & regulation of immune response & 1.9e-02 \\ 
		cellular response to organonitrogen compound & 3.8e-04 & acute-phase response & 2.0e-02 \\ 
		relaxation of cardiac muscle & 7.4e-04 & cellular response to lipopolysaccharide & 2.1e-02 \\ 
		cell surface receptor signaling pathway & 8.4e-04 & negative regulation of relaxation of muscle & 2.1e-02 \\ 
		response to chemical & 9.4e-04 & cellular response to molecule of bacterial origin & 2.5e-02 \\ 
		cellular response to endogenous stimulus & 1.0e-03 & cell migration & 2.6e-02 \\ 
		cellular response to nitrogen compound & 1.2e-03 & striated muscle contraction & 3.0e-02 \\ 
		negative regulation of relaxation of cardiac muscle & 1.4e-03 & extracellular structure organization & 3.0e-02 \\ 
		regulation of cellular component movement & 1.5e-03 & regulation of relaxation of cardiac muscle & 3.0e-02 \\ 
		heart contraction & 3.3e-03 & regulation of interleukin-1-mediated signaling pathway & 3.0e-02 \\ 
		regulation of localization & 3.5e-03 & myeloid leukocyte migration & 3.2e-02 \\ 
		cellular response to stimulus & 3.7e-03 & cellular response to cytokine stimulus & 3.2e-02 \\ 
		response to oxygen-containing compound & 3.8e-03 & signal transduction & 3.6e-02 \\ 
		heart process & 3.8e-03 & movement of cell or subcellular component & 3.6e-02 \\ 
		regulation of blood circulation & 4.7e-03 & cellular response to biotic stimulus & 4.0e-02 \\ 
		blood circulation & 5.4e-03 & system process & 4.2e-02 \\ 
		positive regulation of multicellular organismal process & 5.9e-03 & positive regulation of interferon-gamma production & 4.2e-02 \\ 
		relaxation of muscle & 5.9e-03 & regulation of cell communication & 4.5e-02 \\ 
		circulatory system process & 6.0e-03 & cellular response to acid chemical & 4.8e-02 \\ 
		negative regulation of multicellular organismal process & 6.4e-03 & regulation of signaling & 5.0e-02 \\ 
		cellular response to interleukin-6 & 6.5e-03 &  &  \\ 
	\end{tabularx}
	\smallskip
	\caption{GO terms enriched in the list of 18}
	\label{tab:GO18}
\end{table}


\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Disease} & \textbf{\color{white} q-value} & \textbf{\color{white} Disease} & \textbf{\color{white} q-value} \\
		Endometriosis & 3.14e-10 & Epithelial ovarian cancer & 4.74e-07 \\ 
		Myocardial Infarction & 1.36e-09 & Nephrotic Syndrome & 7.05e-07 \\ 
		Rheumatoid Arthritis & 8.56e-09 & Malignant tumor of cervix & 7.09e-07 \\ 
		Atherosclerosis & 5.34e-08 & Cirrhosis & 8.09e-07 \\ 
		Scleroderma & 5.34e-08 & Ear, patella, short stature syndrome & 8.19e-07 \\ 
		Lymphoma & 5.34e-08 & Cerebrovascular accident & 9.14e-07 \\ 
		Diabetes Mellitus & 5.34e-08 & Fibrosarcoma & 9.29e-07 \\ 
		Arteriosclerosis & 5.34e-08 & Epithelioma & 9.29e-07 \\ 
		Cardiovascular Diseases & 5.52e-08 & Exanthema & 1.18e-06 \\ 
		Diabetes & 5.77e-08 & Cervical cancer & 1.20e-06 \\ 
		Hypertensive disease & 7.17e-08 & Mammary Neoplasms & 1.37e-06 \\ 
		Systemic Scleroderma & 1.13e-07 & Fibrosis, Liver & 1.40e-06 \\ 
		Obesity & 1.13e-07 & Kidney Diseases & 1.40e-06 \\ 
		Diabetic Nephropathy & 1.13e-07 & Colitis & 1.40e-06 \\ 
		Leukemia, Myelocytic, Acute & 2.09e-07 & Liver diseases & 1.55e-06 \\ 
		Elastosis perforans serpiginosa & 2.21e-07 & Hamartoma Syndrome, Multiple & 1.72e-06 \\ 
		Malignant neoplasm of ovary & 2.39e-07 & Hyperglycemia & 1.79e-06 \\ 
		Acute lymphocytic leukemia & 2.90e-07 & Amyloidosis & 1.84e-06 \\ 
		Degenerative polyarthritis & 3.22e-07 & Pancreatic carcinoma & 2.00e-06 \\ 
		Pain & 3.92e-07 & Chronic myeloproliferative disorder & 2.01e-06 \\ 
		Liver neoplasms & 4.53e-07 & Overweight & 2.17e-06 \\ 
		Kidney Failure & 4.53e-07 & Acute Promyelocytic Leukemia & 2.32e-06 \\ 
		Psoriasis & 4.66e-07 & Esophageal carcinoma & 2.51e-06 \\ 
		Liver Cirrhosis & 4.66e-07 & Lupus Erythematosus, Systemic & 2.51e-06 \\ 
		Acute leukemia & 4.73e-07 & Hamman-Rich syndrome & 2.62e-06 \\ 
	\end{tabularx}
	\smallskip
	\caption{Disease form 36 genes}
	\label{tab:diseases}
\end{table}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Biological Process} & \textbf{\color{white} q-value} & \textbf{\color{white} Biological Process} & \textbf{\color{white} q-value}\\
		response to oxygen-containing compound & 3.6e-25 & apoptotic process & 4.0e-16 \\ 
		positive regulation of protein metabolic process & 4.4e-21 & positive regulation of signaling & 5.7e-16 \\ 
		response to organic cyclic compound & 3.0e-20 & regulation of cell proliferation & 6.1e-16 \\ 
		positive regulation of cellular protein metabolic process & 2.4e-19 & cardiovascular system development & 8.6e-16 \\ 
		cellular response to oxygen-containing compound & 1.7e-18 & circulatory system development & 8.6e-16 \\ 
		regulation of programmed cell death & 2.2e-18 & positive regulation of signal transduction & 9.5e-16 \\ 
		response to endogenous stimulus & 2.4e-18 & inflammatory response & 1.8e-15 \\ 
		regulation of cell death & 6.8e-18 & response to cytokine & 3.1e-15 \\ 
		positive regulation of phosphate metabolic process & 1.1e-17 & regulation of system process & 3.5e-15 \\ 
		positive regulation of phosphorus metabolic process & 1.1e-17 & blood circulation & 7.2e-15 \\ 
		protein phosphorylation & 1.1e-17 & positive regulation of intracellular signal transduction & 7.2e-15 \\ 
		regulation of apoptotic process & 2.7e-17 & signal transduction by protein phosphorylation & 7.9e-15 \\ 
		positive regulation of protein modification process & 4.2e-17 & circulatory system process & 7.9e-15 \\ 
		programmed cell death & 5.0e-17 & response to lipid & 8.9e-15 \\ 
		positive regulation of protein phosphorylation & 5.0e-17 & response to wounding & 8.9e-15 \\ 
		regulation of phosphate metabolic process & 5.9e-17 & regulation of transport & 2.2e-14 \\ 
		regulation of phosphorus metabolic process & 7.7e-17 & defense response & 4.4e-14 \\ 
		regulation of protein phosphorylation & 1.5e-16 & blood vessel development & 5.3e-14 \\ 
		response to alcohol & 1.7e-16 & response to hormone & 7.0e-14 \\ 
		positive regulation of phosphorylation & 1.9e-16 & cellular response to endogenous stimulus & 7.4e-14 \\ 
		positive regulation of multicellular organismal process & 1.9e-16 & MAPK cascade & 9.0e-14 \\ 
		positive regulation of cell communication & 2.2e-16 & cellular response to external stimulus & 1.1e-13 \\ 
		regulation of protein modification process & 3.4e-16 & vasculature development & 1.4e-13 \\ 
		regulation of phosphorylation & 3.8e-16 & blood vessel morphogenesis & 2.9e-13 \\ 
		regulation of intracellular signal transduction & 4.0e-16 & angiogenesis & 3.2e-13 \\
	\end{tabularx}
	\smallskip
	\caption{BP 149 genes ToppGenes}
	\label{tab:bp149}
\end{table}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Biological Process} & \textbf{\color{white} q-value} & \textbf{\color{white} Biological Process} & \textbf{\color{white} q-value}\\
		response to oxygen-containing compound & 8.4e-162 & nucleotide metabolic process & 2.7e-91 \\ 
		carboxylic acid metabolic process & 4.5e-157 & nucleoside phosphate metabolic process & 5.3e-91 \\ 
		ion transport & 5.2e-153 & cell-cell signaling & 7.9e-89 \\ 
		organic acid metabolic process & 1.8e-145 & response to drug & 2.2e-88 \\ 
		oxoacid metabolic process & 1.8e-145 & regulation of system process & 2.4e-87 \\ 
		response to endogenous stimulus & 4.1e-134 & response to inorganic substance & 3.2e-87 \\ 
		response to organic cyclic compound & 9.1e-130 & small molecule biosynthetic process & 5.5e-87 \\ 
		response to nitrogen compound & 9.9e-130 & cellular response to oxygen-containing compound & 8.3e-87 \\ 
		response to organonitrogen compound & 1.9e-129 & purine-containing compound metabolic process & 1.3e-85 \\ 
		oxidation-reduction process & 6.3e-124 & chemical homeostasis & 6.8e-85 \\ 
		protein phosphorylation & 1.1e-108 & protein autophosphorylation & 4.9e-83 \\ 
		cation transport & 2.1e-107 & ribose phosphate metabolic process & 9.3e-82 \\ 
		regulation of transport & 2.6e-106 & lipid metabolic process & 1.3e-81 \\ 
		ion transmembrane transport & 4.9e-106 & cation transmembrane transport & 4.1e-81 \\ 
		homeostatic process & 4.8e-103 & response to wounding & 9.4e-80 \\ 
		organophosphate metabolic process & 2.3e-102 & ribonucleotide metabolic process & 3.6e-79 \\ 
		transmembrane transport & 1.5e-99 & purine nucleotide metabolic process & 2.2e-78 \\ 
		response to lipid & 1.2e-97 & organonitrogen compound biosynthetic process & 7.0e-78 \\ 
		response to hormone & 1.5e-97 & anion transport & 4.4e-77 \\ 
		nucleobase-containing small molecule metabolic process & 1.9e-97 & alpha-amino acid metabolic process & 1.1e-76 \\ 
		secretion & 5.4e-97 & purine ribonucleotide metabolic process & 1.3e-76 \\ 
		circulatory system process & 3.4e-96 & inorganic ion transmembrane transport & 1.8e-76 \\ 
		blood circulation & 5.0e-96 & metal ion transport & 2.1e-76 \\ 
		cellular amino acid metabolic process & 4.8e-95 & secretion by cell & 1.7e-73 \\ 
		regulation of ion transport & 2.6e-93 & response to abiotic stimulus & 4.7e-73 \\ 
	\end{tabularx}
	\smallskip
	\caption{BP 2229 genes ToppGenes}
	\label{tab:bp2229}
\end{table}

\end{document}