%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Stylish Article
% LaTeX Template
% Version 2.1 (1/10/15)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Mathias Legrand (legrand.mathias@gmail.com) 
% With extensive modifications by:
% Vel (vel@latextemplates.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------
\PassOptionsToPackage{table}{xcolor}
\PassOptionsToPackage{dvipsnames}{xcolor}

\documentclass[fleqn,10pt]{SelfArx} % Document font size and equations flushed left

\usepackage[english]{babel} % Specify a different language here - english by default
\usepackage{tabularx}
\usepackage[font=normal,labelfont=bf,justification=justified]{caption}
\usepackage{halloweenmath}
\setlength{\parindent}{0pt}

\let\OLDthebibliography\thebibliography
\renewcommand\thebibliography[1]{
	\OLDthebibliography{#1}
	\setlength{\parskip}{0pt}
	\setlength{\itemsep}{0pt plus 0.02ex}
}


%----------------------------------------------------------------------------------------
%	COLUMNS
%----------------------------------------------------------------------------------------

\setlength{\columnsep}{0.55cm} % Distance between the two columns of text
\setlength{\fboxrule}{0.75pt} % Width of the border around the abstract

%----------------------------------------------------------------------------------------
%	COLORS
%----------------------------------------------------------------------------------------

\definecolor{color1}{RGB}{0,0,100} % Color of the article title and sections
\definecolor{color2}{RGB}{0,70,150} % Color of the boxes behind the abstract and headings
\definecolor{color3}{RGB}{250,250,255}

%----------------------------------------------------------------------------------------
%	HYPERLINKS
%----------------------------------------------------------------------------------------

\usepackage{hyperref} % Required for hyperlinks
\hypersetup{hidelinks,colorlinks,breaklinks=true,urlcolor=color2,citecolor=color1,linkcolor=color1,bookmarksopen=false,pdftitle={Title},pdfauthor={Author}}

%----------------------------------------------------------------------------------------
%	ARTICLE INFORMATION
%----------------------------------------------------------------------------------------

\JournalInfo{\color{white} {\huge\color{color3} $\mathwitch*$} The Journal of Computational Witchcraft and Occult Statistics {\huge\color{color3} $\reversemathwitch*$}} % Journal nformation
\Archive{} % Additional notes (e.g. copyright, DOI, review/research article)

\PaperTitle{Identification of putative drug targets for coronary artery disease} % Article title

\Authors{Matteo Ciciani\textsuperscript{1}{\small$^{^\wedge}$}, Enrica Colasurdo\textsuperscript{1}{\small$^{^\wedge}$}, Chiara Mazzoni\textsuperscript{1}{\small$^{^\wedge}$} and Eleonora Nigro\textsuperscript{1}{\small$^{^\wedge}$}} % Authors
\affiliation{\textsuperscript{1}\textit{Quantitative and Computational Biology Master, University of Trento, Italy}}
\affiliation{{\small$^{^\wedge}$}\textit{Equal contributions}}

\Keywords{} % Keywords - if you don't want any simply remove all the text between the curly brackets
\newcommand{\keywordname}{Keywords} % Defines the keywords heading name

%----------------------------------------------------------------------------------------
%	ABSTRACT
%----------------------------------------------------------------------------------------

\Abstract{Drug repurposing is a strategy used to identify new uses for clinically approved drugs. We propose a drug repurposing pipeline that employs NES$^2$RA, a method that allows to identify putative interactions between genes, analyzing gene expression data. This pipeline was applied to identify a list of 32 new potential drug repurposing candidates for Coronary Artery Disease (CAD). Functional enrichment analysis was performed to gain insight on the biological role of the identified genes. Some of the putative targets, namely genes involved in diabetes mellitus and skin diseases, were further investigated, revealing some similarities with genes already targeted for these diseases and CAD.}

%----------------------------------------------------------------------------------------

\begin{document}

\flushbottom % Makes all text pages the same height

\maketitle % Print the title and abstract box

\tableofcontents % Print the contents section

\thispagestyle{empty} % Removes page numbering from the first page

%----------------------------------------------------------------------------------------
%	ARTICLE CONTENTS
%----------------------------------------------------------------------------------------

\section*{Introduction} % The \section*{} command stops section numbering

\addcontentsline{toc}{section}{Introduction} % Adds this section to the table of contents

Drug repurposing is a strategy to identify new therapeutic uses for marketed drugs \cite{Polamreddy}. The use of these procedure has increased in recent years due to new molecular discoveries, new high-throughput technologies and databases. 
s
The traditional drug discovery is difficult and time consuming. Even though investments in biomedical and pharmaceutical research and development has increased significantly in the past 20 years, the annual number of new treatments approved by the US Food and Drug Administration (FDA) has not significantly increased \cite{Feixiong}. This could be due to a lack of well-established  predictive pharmacokinetics and pharmacodynamics approaches, and concerning safety and tolerability profiles for new chemical entities from preclinical studies to clinical trials\cite{Shih}. 

On average, it takes 14 years to be able to have a marketable product, yet most drug development efforts do not pay off.

Drug repurposing can be a good solution to overcome some of the shortcomings of traditional drug discovery. For instance, it can drastically speed up the process, since it relies on previous knowledge and circumvents the FDA approval procedure.

Coronary artery disease (CAD), also known as coronary heart disease (CHD) or ischemic heart disease (IHD), is the most common type of heart disease. It is the leading cause of death in most of the western countries including Italy (World Health Organization data). The condition leads to the formation of a waxy substance called plaque into coronary arteries, decreasing the flow of oxygen-rich blood to the heart. The condition could worsen after the plaque rupture, due to the formation of blood clots that could mostly (or completely) block blood flow through coronary arteries. The disease can weaken the heart muscle and lead to heart failure and arrhythmias. Moreover, it can cause angina, heart attack and sudden cardiac death. \cite{Wong}

CAD can be prevented maintaining a healthy lifestyle with regular exercise and a balanced diet. When this is not enough, the use of medication is needed. Since CAD is a complex disease caused by many factors, different kind of drugs can be used to treat it, among which aspirin is the most known one. However, in the worst cases it is necessary to undergo surgical intervention such as coronary artery bypass surgery (CABG).

Since, as aforementioned, CAD is the leading cause of death in most western countries, we think there is the unmet medical need to broaden the range of drugs effective against this disease.

Currently, there is no standard for \textit{in silico} drug repurposing. In this project we apply NES$^2$RA\cite{NES2RA}, a computational approach that identifies candidate genes that may expand a given gene network. We aim to identify novel putative drug targets for CAD, applying NES$^2$RA to genes genetically associated with CAD. In addition, we aim to test whether NES$^2$RA could be a suitable tool to address drug repurposing.

\section*{Materials and Methods}
\addcontentsline{toc}{section}{Materials and Methods} % Adds this section to the table of contents

\subsection*{Platforms and Data Sources}
\addcontentsline{toc}{subsection}{Platforms and Data Sources} % Adds this section to the table of contents


\subsubsection*{NES$^2$RA and BOINC}
NES$^2$RA is a method based on the PC-algorithm \cite{PC-alg} and can be summarized in three steps. First, data undergo subsetting, then the skeleton of the PC algorithm is run and finally results are aggregated, assigning to each isoform in the expansion lists the corresponding relative frequency score.

\setlength\emergencystretch{\textwidth}
To facilitate the most computationally demanding step of NES$^2$RA, i.e. running the skeleton of the PC algorithm, the single gene expansions were submitted to gene@home \cite{realBoinc} BOINC (Berkeley Open Infrastructure for Network Computing) project, hosted by the TN-Grid platform. The platform allows to distribute the computational burden among committed volunteers' machines, in order to reduce the running time and memory usage as much as possible.

This gene@home project was previously set up for the network expansion of genes whose expression data came from the FANTOM5 Database \cite{fantom}. On the BOINC platform, alongside the original FANTOM5 expression data, a filtered version is available. The latter was built selecting, among the isoforms encoded by the same gene, the ones that differ mostly in terms of expression profile, presumably corresponding to different biological functions. We chose to use the filtered version, in order to cut down on the computational load of subsequent analyses.

NES$^2$RA was run with default parameters: $\alpha=0.05$, tile size $=1000$ and 2000 iterations.

\subsubsection*{On-line resources}
\emph{Open Targets} is a platform which integrates public domain data to enable drug target identification and prioritisation\cite{open, OpenTargetPlatform}. This resource was used to select genes genetically associated with CAD disease, to identify drug targets and to perform functional enrichment analysis on the final results.

The \emph{DrugBank} database is a bioinformatics and cheminformatics resource that combines detailed drug data with comprehensive drug target information \cite{drugbank}. This resource was used to retrieve all the genes already targeted by FDA approved drugs.

\emph{ToppGene} is a portal for gene list enrichment analysis and candidate gene prioritization based on functional annotations and protein interaction networks \cite{toppgene}. This resource was used to perform functional enrichment analysis on the final results.

\subsection*{Pipeline}
\addcontentsline{toc}{subsection}{Pipeline} % Adds this section to the table of contents

Our pipeline can be summarized in  6 steps, that are briefly outlined below and are represented in Fig. \ref{Fig:pipe}.

\begin{itemize}[leftmargin=*]
	\item[1.] Starting from a list of 46 genes genetically associated with CAD, obtained from Open Targets, single gene expansions are performed using NES$^2$RA.
	\item[2.] The resulting expansion lists are aggregated, obtaining a score for each isoform present in the expansion lists. This score is meant to recapitulate the overall interaction of an isoform with the list of CAD genes. Isoforms are then filtered according to the genetic association scores of the CAD genes they interact with.
	\item[3.] The resulting list of isoforms is converted in a list of genes. Among these genes, we select those that are targets of clinically approved drugs. The result of this step is a list of preliminary putative drug repurposing targets.
	\item[4.] The list of preliminary targets is further analyzed, performing single gene expansions using NES$^2$RA.
	\item[5.] The Weighted Jaccard Similarity (WJS) \cite{WGS} is used to compare a ranked list of genes genetically associated with CAD, obtained from Open Targets, with the output of NES$^2$RA.
	\item[6.] A statistical analysis of the WJS scores is employed in order to identify which genes present a significant interaction with the list of CAD genes. The result of this step is a final list of putative drug repurposing targets.
\end{itemize}

The rationale of steps 4, 5 and 6 is the identification of genes, among the preliminary targets, that present a strong interaction with CAD associated genes. This analysis allows us to have a larger confidence on the final list of putative targets, since their interactions with CAD genes has been tested twice (they were found in the expansions of CAD genes and CAD genes are present in their expansions).

Each step of the pipeline is described in further detail below.

\subsubsection*{Selection of CAD genes}
Open Target allows to search for genes associated with a given disease. We downloaded from the website the list of 643 genes genetically associated with CAD, ranked them based on the genetic association score and focused on the first 100.

The FANTOM5 database is built on gene isoforms and we were provided with an annotation file to perform the mapping of a gene into its corresponding isoforms.

Some of the downloaded CAD genes were not present in the annotation, therefore we had to exclude them aprioristically. Given unlimited time, we would have expanded all the selected 100 genes, yet we had to restrain our search to 41 of them, aside from the 5 that had already been expanded by previous runs of the gene@home BOINC project.

\subsubsection*{NES$^{\textbf{2}}$RA expansion of CAD associated genes}
The 41 CAD genetically associated genes, totaling to 183 isoforms, were turned into one gene network files as inputs for NES$^2$RA.
The algorithm outputs a series of expansion networks, each of which is a list of nodes ranked on the reported score, which is a measure of the strength of the supposed correlation between that node and the input one. 

\subsubsection*{Score aggregation and filtering}
The output lists were merged on the isoforms, summing their scores in order to rank them based on how many times each isoform appeared in the various expansion lists.

Due to the large number of isoforms present in the expansion lists, we performed a selection on the isoforms. For each expansion list $\ell$, only the first $N_\ell$ isoforms were selected, where $N_\ell=max\{5;\,100-R_\ell\}$, and $R_\ell$ is the rank, based on genetic association, of the gene whose isoform was expanded to obtain the list $\ell$. 

The resulting isoforms were ranked according to the aggregated score described previously. Then, the ranked list of isoforms was converted in a ranked list of genes. The rank of a gene was obtained as the minimum of the ranks of its corresponding isoforms. 

\subsubsection*{Selection of target genes}

The Open Targets Python API was used to query which of these genes are known to be targets of clinically approved drugs. We obtained a list of 149 target genes, which are the only ones that are considered in the following analyses.

\subsubsection*{NES$^{\textbf{2}}$RA expansion of target genes}

In order to identify a subset of genes with a strong association with CAD, we selected the isoforms corresponding to these 149 target genes and we performed single isoform expansions using NES$^2$RA. For each gene, only the isoforms that passed the previous filter were considered.

\begin{figure}
	\includegraphics[width=1.1\linewidth, trim = 1cm 0cm 0.8cm 0.4cm, clip = true]{pipeline.png}
	\caption{Visual representation of the pipeline used. Color legend: blue = databases, yellow = data files, red = data processing steps.}
	\label{Fig:pipe}
\end{figure}

\subsubsection*{Comparison of expanded lists with CAD genes}

We expect NES$^2$RA expansions of genes that have strong interaction with CAD genes to have a large overlap with the list of CAD genes we started from. In order to quantify the overlap between the ranked list of CAD genes and the ranked expansion lists obtained in the previous step, we used the Weighted Jaccard Similarity (WJS), which we define below.\medskip

\noindent
\textit{Definition: given two weighted list of items, $\rho$ and $\sigma$, of equal length $N$, their Weighted Jaccard Similarity, $WJS(\rho, \sigma)$, is defined as:}

$$
WJS(\rho, \sigma) = \dfrac{\sum_{i=1}^Nmin(\rho_i,\sigma_i)}{\sum_{i=1}^Nmax(\rho_i,\sigma_i)}
$$

\noindent
\textit{where $\rho_i$ and $\sigma_i$ are the weights corresponding to the same item $i$.}\medskip

In our analysis, the weight of a feature $i$ (gene or isoform) in a ranked list $\rho$ is computed as $length(\rho) - rank_{\rho}(i) + 1$. This allows to assign large weights to high ranking features.

In principle, the lists of features do not contain the same number of elements. To solve this, we add the missing elements of one list to the other (and vice versa) as ties in the last position of the ranking.

To compute the WJS between the CAD genes and each expansion list, we have to either convert genes into isoforms or vice versa. We explored both options:

\begin{itemize}[leftmargin=*]
	\item \textit{Convert NES$^2$RA isoforms into genes}: Given a list of isoforms obtained with NES$^2$RA, we can convert it in a list of genes, ranking them by the largest relative frequency of their isoforms.
	\item \textit{Convert CAD genes into isoforms}: The ranked list of CAD genes was converted into a ranked list of isoforms, treated as ties in the ranking.
\end{itemize}

We then computed the WJS scores for both cases.

\subsubsection*{Significance analysis} 

The scores obtained from the WJS are not directly comparable between them, since they depend on the length of the lists. In order to obtain values that can be compared directly, we used a permutation approach to estimate a set of score distributions. For each length present, we generated 2000 random lists of genes or isoforms, we computed the WJS and we generated the score distribution associated to that length. Then, we used these distributions to compute p-values. The Benjamini-Hochberg correction was used to adjust p-values for multiple hypothesis testing \cite{BH}. A significance level of 0.05 was used to identify statistically significant WJS scores. 

\subsection*{Enrichment analysis}
\addcontentsline{toc}{subsection}{Enrichment analysis} % Adds this section to the table of contents

In order to gain insight on the biological role of identified genes, we performed functional enrichment analysis. We used both Open Targets and ToppGene to perform enrichment analysis against the Biological Processes from Gene Ontology. In addition, we used ToppGene to perform enrichment analysis against disease databases (Clinical Variations, DisGeNET BeFree, DisGeNET Curated, GWAS and OMIM).

\section*{Results}
\addcontentsline{toc}{section}{Results} % Adds this section to the table of contents

\subsubsection*{Intermediate results}
Step 3 of the pipeline (see Materials and Methods) produced a list of 2043 genes, of which 149 are targets of clinically approved drugs (reported in Tab. \ref{tab:149genes}). For both lists, we employed ToppGene to perform enrichment analysis against the Biological Processes from Gene Ontology. The 50 most significant terms are reported in Tab. \ref{tab:bp2043} and Tab. \ref{tab:bp149}.

Since genes involved in cardiovascular diseases are highly represented among genes targeted by drugs, we may expect the enrichment analysis of the second set to present a bias. Therefore, we retrieved the list of all genes targeted by approved drugs from DrugBank and we performed an enrichment analysis on it (Tab. \ref{tab:bp2229}). We compared the results of this analysis with the results of the previous one in order to evaluate the bias.

\subsubsection*{Final results}

Due to time constraints, only 173 isoform expansions (step 4 of the pipeline) were completed, over a total of 216.

In the last step of the pipeline, we obtained 36 genes from the first approach (converting isoforms to genes) and 18 genes from the second approach (converting genes to isoforms). Remarkably, there is a striking overlap (16 genes) between the two lists. These results are reported in Tab. \ref{tab:sign36} and \ref{tab:sign18}.

We used Open Targets to identify which drugs target the 36 genes identified. In Tab. \ref{tab:drugs} we reported only the drugs at phase IV of clinical trial.

For both lists, we employed Open Targets to perform enrichment analysis against the Biological Processes from Gene Ontology and ToppGene to perform enrichment analysis against disease databases. The results are reported in Tab. \ref{tab:GO36_1}, Tab. \ref{tab:GO36_2}, Tab. \ref*{tab:GO18} (for Biological Processes) and Tab. \ref{tab:diseases36}, Tab. \ref{tab:diseases18} (for the diseases).

Among the 36 genes, 19 were already associated with CAD. 4 of them are also target of drugs used to treat CAD. The list of genes and their association levels are reported in Fig. \ref{fig:19CAD}.

\begin{figure}[ht]
	\includegraphics[width=\linewidth]{already_in}
	\smallskip
	\caption{Putative targets already associated with CAD. Four of them are also targets of drugs used to treat CAD. The association scores were color coded, with white being the lowest value and blue being the largest.}
	\label{fig:19CAD}
\end{figure}

\renewcommand{\arraystretch}{1.1}
\newcolumntype{R}{>{\raggedleft\arraybackslash}X}
\begin{table}[!t]
	\centering
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\linewidth}{lRRc}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Gene} & \textbf{\color{white} Number of isoforms} & \textbf{\color{white} q-value} & \textbf{\color{white}
			CAD gene}\\
	CASP7 &   1 & 0 & No \\ 
	NDUFA4L2 &   1 & 0 & Yes \\ 
	INSR &   1 & 0 & Yes \\ 
	PDE4B &   2 & 0 & Yes \\ 
	IL1R1 &   1 & 0 & No \\ 
	CTGF &   2 & 0 & No \\ 
	FN1 &   8 & 0 & Yes \\ 
	COL1A1 &   1 & 0 & Yes \\ 
	PDE4D &   3 & 0 & Yes \\ 
	COL4A1 &   4 & 0 & Yes \\ 
	COL4A2 &   6 & 0 & Yes \\ 
	AVPR1A &   1 & 2.9e-03 & No \\ 
	PIK3CA &   1 & 2.9e-03 & No \\ 
	CDK14 &   1 & 2.6e-02 & No \\ 
	IL6 &   1 & 3.2e-02 & Yes \\ 
	KCNJ2 &   1 & 4.6e-02 & No \\
	\hline 
	JAK1 &   1 & 0 & No \\ 
	RARA &   1 & 0 & Yes \\ 
	TLR4 &   2 & 0 & Yes \\ 
	PIK3CG &   1 & 1.2e-02 & Yes \\ 
	FNTB &   1 & 1.6e-02 & No \\ 
	CSF1 &   1 & 1.7e-02 & Yes \\ 
	TSG101 &   1 & 1.7e-02 & No \\ 
	F5 &   1 & 2.3e-02 & No \\ 
	TGFB1 &   2 & 2.7e-02 & Yes \\ 
	ANPEP &   1 & 2.9e-03 & Yes \\ 
	DNMT3A &   1 & 2.9e-03 & No \\ 
	TNFRSF1A &   1 & 2.9e-03 & Yes \\ 
	FLT1 &   1 & 3.5e-02 & Yes \\ 
	SEBOX &   1 & 3.8e-02 & No \\ 
	COL5A3 &   1 & 4.4e-02 & No \\ 
	GAA &   1 & 4.4e-02 & No \\ 
	FGA &   1 & 4.9e-02 & Yes \\ 
	NAMPT &   1 & 5.3e-03 & Yes \\ 
	IFNAR2 &   1 & 7.5e-03 & No \\ 
	PTGER4 &   1 & 7.5e-03 & No \\ 
	\end{tabularx}
	\smallskip
	\caption{List of 36 genes resulting from the last step of the pipeline (first approach). The table shows the number of significant isoforms and the minimum q-value associated with that gene.}
	\label{tab:sign36}
\end{table}

\begin{table}[!t]
	\centering
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\linewidth}{lRRc}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Gene} & \textbf{\color{white} Number of isoforms} & \textbf{\color{white} q-value} & \textbf{\color{white}
			CAD gene}\\
		CASP7 &   1 & 0 & No \\ 
		NDUFA4L2 &   1 & 0 & Yes \\ 
		INSR &   1 & 4.0e-02 & Yes \\ 
		PDE4B &   1 & 1.7e-02 & Yes \\ 
		IL1R1 &   1 & 0 & No \\ 
		CTGF &   1 & 0 & No \\ 
		FN1 &   8 & 0 & Yes \\ 
		COL1A1 &   1 & 0 & Yes \\ 
		PDE4D &   2 & 7.5e-03 & Yes \\ 
		COL4A1 &   4 & 0 & Yes \\ 
		COL4A2 &   3 & 0 & Yes \\ 
		AVPR1A &   1 & 4.1e-03 & No \\ 
		PIK3CA &   1 & 3.9e-02 & No \\ 
		CDK14 &   1 & 2.2e-02 & No \\ 
		IL6 &   1 & 1.7e-02 & Yes \\ 
		KCNJ2 &   1 & 1.1e-02 & No \\
		\hline 
		HAMP &   1 & 3.7e-02 & No \\ 
		FGG &   1 & 4.7e-02 & No \\ 
	\end{tabularx}
	\smallskip
	\caption{List of 18 genes resulting from the last step of the pipeline (second approach). The table shows the number of significant isoforms and the minimum q-value associated with that gene.}
	\label{tab:sign18}
\end{table}

\section*{Discussion}
\addcontentsline{toc}{section}{Discussion} % Adds this section to the table of contents

NES$^2$RA one gene expansions are meant to explore putative causal relationships between genes. Since NES$^2$RA retrieves up to several thousands genes for each expansion, we have to apply aggregation and filtering techniques in order to extract meaningful biological information from the expansion lists. One of such techniques is the comparison of expansion lists of putative targets with the list of genes already known to be associated with CAD, which allows to identify target genes that are more likely to interact directly with CAD genes. It is worth mentioning that this approach focuses only on the identification of a subset of all the possible putative targets, namely those that interact with CAD genes at the transcriptional level.

\subsubsection*{Discussion of intermediate results}

In order to critically evaluate the biological relevance of the results of 
NES$^2$RA, we chose to perform enrichment analysis. We expect to find Gene Ontology terms related to the cardiovascular systems. The absence of terms related to the cardiovascular systems would have introduces significant skepticism on the biological validity of the results.

The enrichment analysis applied to the list of 2043 genes resulting from step 3 of the pipeline yielded a large variety of biological processes, among which some related to the circulatory system, as \emph{vasculature development} and \emph{blood vessel development}.

To assess whether the subset of 149 target genes is also related to the cardiovascular system, we performed an enrichment analysis that retrieved a larger number of terms associated with the circulatory system, such as \emph{blood circulation}, \emph{circulatory system process} and \emph{angiogenesis}. This suggests that NES$^2$RA was able to retrieve a group of target genes that directly interact with CAD genes and appear to be involved in the cardiovascular system.

As stated in the Results section, this last result could reflect a bias introduced by the selection of drug targets. To assess this we performed enrichment analysis on all the genes targeted by approved drugs. Nevertheless, according to the results coming from ToppGene, there is no evident bias toward the cardiovascular system.

\subsubsection*{Discussion of final results}

Considering the results of the final step of the pipeline (the lists of 36 and 18 genes), a cautious approach would preferentially draw conclusions from the more stringent set, both due to the large overlap and the absence of information loss (resulting from the avoidance of the passage from isoform to gene). In the absence of experimental validations of the findings of the two methods, we deemed these arguments to be insufficient to dismiss the broader selection, carrying out a comparison of the two instead.

The two lists of genes were subjected to enrichment analysis against GO biological processes. As expected we found multiple process related to signal transduction pathways, which are typical drug targets.

In order to gain insight on the genes found, an enrichment analysis against the databases of diseases was performed using ToppGene. As expected, we retrieved similar terms between the two lists that can be grouped by different categories: metabolic diseases, autoimmune diseases (particularly skin and joints), skin diseases, diseases of female reproductive organs, cardiovascular diseases and kidney diseases. We decided to analyze in further detail the relationship of some of these diseases with CAD, using the literature.

\subsubsection*{Diabetes Mellitus}

The relation between diabetes mellitus (DM) and cardiovascular diseases is well known. For DM, CAD is a major determinant of the long-term prognosis among patients. DM is associated with a 2 to 4-fold increased mortality risk from heart disease \cite{diabetes}.

Metformin is an example of a drug used to treat both CAD and DM. This drug targets NDUFAL4L2, one of the genes that was retrieved by our pipeline (Fig. \ref{fig:19CAD}). \cite{metformin} Considering this result, we propose to repurpose drugs used to treat DM for CAD. From our analyses, the genes INSR and GAA seem to be promising target for drug repurposing.

INSR (Insulin Receptor) mediates the pleiotropic actions of insulin. Binding of insulin leads to phosphorylation of several intracellular substrates, including insulin receptor substrates and other signaling intermediates. This gene has a strong  genetic association with CAD and is targeted by insulin. Given the broad range of processes regulated by this receptor and its association with CAD, it could be a good candidate for drug repurposing.

GAA  (Glucosidase Alpha, Acid) is a protein coding gene essential for the degradation of glycogen in lysosomes. This gene is not associated with CAD, but it was identified by our pipeline. It is targeted by Miglitol, a drug that acts by inhibiting the ability of the patient to breakdown complex carbohydrates into glucose.

\subsubsection*{Skin conditions}

Among the 36 genes, PDE4B already has a phase IV drug for CAD (namely, Dypiridamole), alongside approved drugs for some skin conditions, such as atopic eczema and alopecia. This link leads us to suggest that the several drugs acting on skin conditions, targeting the gene RARA (one of our 36 genes), could be investigated through clinical trials for CAD. RARA is very weakly associated with CAD, therefore any positive therapeutic confirmation would be particularly significant for the assessment of the efficacy of NES$^2$RA for drug repurposing.

On a side note, this kind of information can be used for cross-repurposing aims. For example, Pentoxifylline, currently a phase II drug for CAD disease, is reported in the literature to be a drug with wide spectrum applications in dermatology, although it has not being investigated thoroughly for these applications. \cite{pento} This could potentially foster the connection between CAD and skin conditions.

\section*{Conclusions}
\addcontentsline{toc}{section}{Conclusions} % Adds this section to the table of contents

The application of drug repurposing is an effective strategy to reduce the costs of the drug development process and has successfully applied in recent years, however the lack of standard computational pipelines still hinders the progress in this field.

Here we presented a novel drug repurposing pipeline, which has lead to the identification of 36 putative drug targets, 4 of which are already known to be used for CAD. In addition we explore in further details the relationship between CAD and pathologies like DM and skin diseases, proposing new potential drug targets.

Needless to say, a large amount of experimental data would need to be gathered and analyzed in order to draw conclusions on the validity of the methods, namely NESSRA and our pipeline, and of our findings.

\phantomsection
%----------------------------------------------------------------------------------------
%	REFERENCE LIST
%----------------------------------------------------------------------------------------
\phantomsection
%\bibliographystyle{unsrt}
%\bibliography{biblio}
\begin{thebibliography}{1}
	
	%Formato PLOS	
	
	%1
	\bibitem{Polamreddy}Polamreddy P, Gattu N. The drug repurposing landscape from 2012 to 2017: evolution, challenges, and possible solutions. Drug Discov Today. 2018; doi:10.1016/j.drudis.2018.11.022.		
	
	%2
	\bibitem{Feixiong}Cheng F, Desai RJ, Handy DE, Wang R, Schneeweiss S, Barabási A-L, et al. Network-based approach to prediction and population-based validation of in silico drug repurposing. Nat Commun. 2018;9: 2691.
	
	%3
	\bibitem{Shih}Shih H-P, Zhang X, Aronov AM. Drug discovery effectiveness from the standpoint of therapeutic mechanisms and indications. Nat Rev Drug Discov. 2018;17: 78.
	
	%4
	\bibitem{Wong}Wong ND. Epidemiological studies of CHD and the evolution of preventive cardiology. Nat Rev Cardiol. 2014;11: 276–289.
	
	%5	problema non trova quello del 2018
	\bibitem{NES2RA}Asnicar F, Masera L, Coller E, Gallo C, Sella N, Tolio T, et al. NES2RA. Int J High Perform Comput Appl. 2016;32: 380–392.
	
	%6 
	\bibitem{PC-alg}Spirtes P, Glymour CN. An Algorithm for Fast Recovery of Sparse Causal Graphs. 1990.
	
	%7
	\bibitem{realBoinc}Anderson DP. BOINC: A System for Public-Resource Computing and Storage. Fifth IEEE/ACM International Workshop on Grid Computing. doi:10.1109/grid.2004.14

	%8
	\bibitem{fantom}Noguchi S, Arakawa T, Fukuda S, Furuno M, Hasegawa A, Hori F, et al. FANTOM5 CAGE profiles of human and mouse samples. Sci Data. 2017;4: 170112.
	
	
	%9 
	\bibitem{open}Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, et al. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 2017;45: D985–D994.
	
	%10
	\bibitem{OpenTargetPlatform}Carvalho-Silva D, Pierleoni A, Pignatelli M, Ong C, Fumis L, Karamanis N, et al. Open Targets Platform: new developments and updates two years on. Nucleic Acids Res. 2018; doi:10.1093/nar/gky1133 
	
	%11
	\bibitem{drugbank}Wishart DS, Feunang YD, Guo AC, Lo EJ, Marcu A, Grant JR, et al. DrugBank 5.0: a major update to the DrugBank database for 2018. Nucleic Acids Res. 2017;46: D1074–D1082.
	
	%12
	\bibitem{toppgene}Chen J, Bardes EE, Aronow BJ, Jegga AG. ToppGene Suite for gene list enrichment analysis and candidate gene prioritization. Nucleic Acids Res. 2009;37: W305–11.
	
	%13
	\bibitem{WGS}Ioffe S. Improved Consistent Sampling, Weighted Minhash and L1 Sketching. 2010 IEEE International Conference on Data Mining. 2010. doi:10.1109/icdm.2010.80

	%14
	\bibitem{BH}Benjamini Y, Hochberg Y. Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J R Stat Soc Series B Stat Methodol. 1995;57: 289–300.
	
	%15
	\bibitem{diabetes} Aronson D, Edelman ER. Coronary Artery Disease and Diabetes Mellitus. Heart Fail Clin. 2016;12: 117–133.

	%16
	\bibitem{metformin} Pryor R, Cabreiro F. Repurposing metformin: an old drug with new tricks in its binding pockets. Biochem J. 2015;471: 307–322.
	
	%17
	\bibitem{pento} Çakmak SK, Çakmak A, Gönül M, Kiliç A, Gül Ü. Pentoxifylline use in dermatology. Inflamm Allergy Drug Targets. 2012;11: 422–432
	
	
\end{thebibliography}

%----------------------------------------------------------------------------------------
%	SUPPLEMENTARY MATERIAL
%----------------------------------------------------------------------------------------

\pagebreak
\onecolumn

\section*{SUPPLEMENTARY MATERIAL}

\addcontentsline{toc}{section}{Supplementary Material} % Adds this section to the table of contents

\setcounter{table}{0}
\renewcommand{\thetable}{S\arabic{table}}
\renewcommand{\arraystretch}{1.2}

\begin{table}[!htb]
	\centering
	\scriptsize
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\textwidth}{XXXXXX}
		NDUFA4L2 & JAK1 & CDK16 & SRD5A1 & PRKCD & FLT4 \\ 
		FN1 & KCNJ2 & PIM2 & FGFR4 & ABCA1 & DHODH \\ 
		COL4A1 & FDPS & COL27A1 & TUBA1B & ERBB3 & FNTB \\ 
		COL4A2 & CPT2 & ALOX5 & PIK3CG & NR3C1 & TNFSF12 \\ 
		CTGF & LAMB3 & CD70 & GABBR1 & HGF & TGFB3 \\ 
		F5 & PCSK9 & GSK3B & PDE2A & BIRC2 & KCNH8 \\ 
		SEBOX & FLT1 & SLC29A1 & MMP9 & ERAP1 & EGLN2 \\ 
		TGFB1 & VEGFB & PIM3 & PSCA & OPRL1 & METAP2 \\ 
		F2 & POLD1 & CSF1 & CALM1 & GSTP1 & LAMA2 \\ 
		TLR4 & IL1R1 & CASP7 & PDE8B & PRKACA & NDUFA13 \\ 
		COL5A3 & XPNPEP2 & GAA & DNMT3A & RAMP2 & MAPKAPK5 \\ 
		COL1A1 & NR1H3 & PDE4B & IL6 & HSP90AA1 & CD276 \\ 
		FGA & NFE2L2 & PTGS2 & EPHA2 & INSR & HDAC5 \\ 
		FOLR1 & GBA & NCSTN & PIK3CA & MAP3K9 & DRD2 \\ 
		DHCR24 & PLA2G2A & NDUFS2 & TNFRSF8 & TNFSF13 & NTRK1 \\ 
		GLP2R & ACAT1 & TSG101 & LTB4R & CASP8 & PIK3R3 \\ 
		TNFRSF1A & FDFT1 & PLK1 & VDR & CACNG4 & MALT1 \\ 
		AVPR1A & HLA-DRB1 & NDUFB7 & EEF2 & NFE2 & SLC6A9 \\ 
		FGG & IFNGR2 & BCL2L1 & SCN9A & LTBR & HDAC6 \\ 
		NAMPTL & PLA2G7 & IDH2 & PTH1R & EPHB6 & PTGER4 \\ 
		SRD5A3 & CHEK2 & PSMA4 & P4HB & GART & KCNK10 \\ 
		PDE4D & ATP1B3 & ENG & CFD & KCNH2 & CD44 \\ 
		HAMP & ANPEP & EPHA1 & ARAF & S1PR5 & P4HTM \\ 
		RARA & COL6A3 & TNFRSF10B & SIGMAR1 & TUBB & MET \\ 
		NAMPT & COL11A2 & CDK14 & IFNAR2 & KCNC3 &  \\ 
	\end{tabularx}
	\smallskip
	\caption{List of 149 preliminary target genes resulting from step 3 of the pipeline.}
	\label{tab:149genes}
\end{table}

\begin{table}[!htb]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Biological Process} & \textbf{\color{white} q-value} & \textbf{\color{white} Biological Process} & \textbf{\color{white} q-value}\\
		organic substance catabolic process & 4.4e-11 & protein catabolic process & 1.2e-07 \\ 
		positive regulation of biosynthetic process & 4.4e-11 & regulation of protein catabolic process & 1.2e-07 \\ 
		regulation of intracellular signal transduction & 1.2e-10 & positive regulation of cellular protein metabolic process & 1.6e-07 \\ 
		positive regulation of cellular biosynthetic process & 1.5e-10 & regulation of cell proliferation & 1.8e-07 \\ 
		positive regulation of gene expression & 4.1e-10 & positive regulation of molecular function & 2.4e-07 \\ 
		response to oxygen-containing compound & 4.6e-10 & alcohol metabolic process & 2.8e-07 \\ 
		positive regulation of nucleic acid-templated transcription & 4.6e-10 & negative regulation of RNA metabolic process & 2.9e-07 \\ 
		positive regulation of transcription, DNA-templated & 4.6e-10 & lipid metabolic process & 3.3e-07 \\ 
		positive regulation of macromolecule biosynthetic process & 1.2e-09 & \textbf{blood vessel development} & 6.3e-07 \\ 
		positive regulation of RNA biosynthetic process & 1.2e-09 & regulation of proteolysis & 7.0e-07 \\ 
		cellular catabolic process & 1.2e-09 & negative regulation of biosynthetic process & 7.1e-07 \\ 
		positive regulation of RNA metabolic process & 1.6e-09 & regulation of kinase activity & 9.1e-07 \\ 
		positive regulation of nitrogen compound metabolic process & 2.7e-09 & negative regulation of macromolecule biosynthetic process & 9.7e-07 \\ 
		positive regulation of protein metabolic process & 2.7e-09 & multi-organism cellular process & 9.7e-07 \\ 
		positive regulation of nucleobase-containing compound metabolic process & 3.3e-09 & regulation of protein kinase activity & 1.1e-06 \\ 
		negative regulation of gene expression & 1.1e-08 & negative regulation of response to stimulus & 1.1e-06 \\ 
		regulation of response to stress & 2.0e-08 & response to endogenous stimulus & 1.1e-06 \\ 
		macromolecule catabolic process & 4.6e-08 & cell cycle & 1.1e-06 \\ 
		\textbf{vasculature development} & 4.6e-08 & negative regulation of cellular macromolecule biosynthetic process & 1.1e-06 \\ 
		regulation of catabolic process & 5.2e-08 & symbiont process & 1.1e-06 \\ 
		negative regulation of nitrogen compound metabolic process & 5.4e-08 & interspecies interaction between organisms & 1.1e-06 \\ 
		\textbf{cardiovascular system development} & 8.4e-08 & negative regulation of cellular biosynthetic process & 1.1e-06 \\ 
		\textbf{circulatory system development} & 8.4e-08 & negative regulation of transcription, DNA-templated & 1.1e-06 \\ 
		negative regulation of nucleobase-containing compound metabolic process & 8.7e-08 & regulation of protein serine/threonine kinase activity & 1.3e-06 \\ 
		cellular response to oxygen-containing compound & 1.2e-07 & negative regulation of RNA biosynthetic process & 1.6e-06 \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against GO Biological processes of the list of 2043 genes, resulting from step 3 of the pipeline. For each term, q-values, resulting from Benjamini-Hochberg correction, are reported. Only the first 50 most significant terms are reported.}
	\label{tab:bp2043}
\end{table}

\begin{table}[!ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Biological Process} & \textbf{\color{white} q-value} & \textbf{\color{white} Biological Process} & \textbf{\color{white} q-value}\\
		response to oxygen-containing compound & 3.6e-25 & apoptotic process & 4.0e-16 \\ 
		positive regulation of protein metabolic process & 4.4e-21 & positive regulation of signaling & 5.7e-16 \\ 
		response to organic cyclic compound & 3.0e-20 & regulation of cell proliferation & 6.1e-16 \\ 
		positive regulation of cellular protein metabolic process & 2.4e-19 & \textbf{cardiovascular system development} & 8.6e-16 \\ 
		cellular response to oxygen-containing compound & 1.7e-18 & \textbf{circulatory system development} & 8.6e-16 \\ 
		regulation of programmed cell death & 2.2e-18 & positive regulation of signal transduction & 9.5e-16 \\ 
		response to endogenous stimulus & 2.4e-18 & inflammatory response & 1.8e-15 \\ 
		regulation of cell death & 6.8e-18 & response to cytokine & 3.1e-15 \\ 
		positive regulation of phosphate metabolic process & 1.1e-17 & regulation of system process & 3.5e-15 \\ 
		positive regulation of phosphorus metabolic process & 1.1e-17 & \textbf{blood circulation} & 7.2e-15 \\ 
		protein phosphorylation & 1.1e-17 & positive regulation of intracellular signal transduction & 7.2e-15 \\ 
		regulation of apoptotic process & 2.7e-17 & signal transduction by protein phosphorylation & 7.9e-15 \\ 
		positive regulation of protein modification process & 4.2e-17 & \textbf{circulatory system process} & 7.9e-15 \\ 
		programmed cell death & 5.0e-17 & response to lipid & 8.9e-15 \\ 
		positive regulation of protein phosphorylation & 5.0e-17 & \textbf{response to wounding} & 8.9e-15 \\ 
		regulation of phosphate metabolic process & 5.9e-17 & regulation of transport & 2.2e-14 \\ 
		regulation of phosphorus metabolic process & 7.7e-17 & defense response & 4.4e-14 \\ 
		regulation of protein phosphorylation & 1.5e-16 & \textbf{blood vessel development} & 5.3e-14 \\ 
		response to alcohol & 1.7e-16 & response to hormone & 7.0e-14 \\ 
		positive regulation of phosphorylation & 1.9e-16 & cellular response to endogenous stimulus & 7.4e-14 \\ 
		positive regulation of multicellular organismal process & 1.9e-16 & MAPK cascade & 9.0e-14 \\ 
		positive regulation of cell communication & 2.2e-16 & cellular response to external stimulus & 1.1e-13 \\ 
		regulation of protein modification process & 3.4e-16 & \textbf{vasculature development} & 1.4e-13 \\ 
		regulation of phosphorylation & 3.8e-16 & \textbf{blood vessel morphogenesis} & 2.9e-13 \\ 
		regulation of intracellular signal transduction & 4.0e-16 & \textbf{angiogenesis} & 3.2e-13 \\
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against GO Biological processes of the list of 149 preliminary target genes, resulting from step 3 of the pipeline. For each term, q-values, resulting from Benjamini-Hochberg correction, are reported. Only the first 50 most significant terms are reported.}
	\label{tab:bp149}
\end{table}

\begin{table}[!htb]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Biological Process} & \textbf{\color{white} q-value} & \textbf{\color{white} Biological Process} & \textbf{\color{white} q-value}\\
		response to oxygen-containing compound & 8.4e-162 & nucleotide metabolic process & 2.7e-91 \\ 
		carboxylic acid metabolic process & 4.5e-157 & nucleoside phosphate metabolic process & 5.3e-91 \\ 
		ion transport & 5.2e-153 & cell-cell signaling & 7.9e-89 \\ 
		organic acid metabolic process & 1.8e-145 & response to drug & 2.2e-88 \\ 
		oxoacid metabolic process & 1.8e-145 & regulation of system process & 2.4e-87 \\ 
		response to endogenous stimulus & 4.1e-134 & response to inorganic substance & 3.2e-87 \\ 
		response to organic cyclic compound & 9.1e-130 & small molecule biosynthetic process & 5.5e-87 \\ 
		response to nitrogen compound & 9.9e-130 & cellular response to oxygen-containing compound & 8.3e-87 \\ 
		response to organonitrogen compound & 1.9e-129 & purine-containing compound metabolic process & 1.3e-85 \\ 
		oxidation-reduction process & 6.3e-124 & chemical homeostasis & 6.8e-85 \\ 
		protein phosphorylation & 1.1e-108 & protein autophosphorylation & 4.9e-83 \\ 
		cation transport & 2.1e-107 & ribose phosphate metabolic process & 9.3e-82 \\ 
		regulation of transport & 2.6e-106 & lipid metabolic process & 1.3e-81 \\ 
		ion transmembrane transport & 4.9e-106 & cation transmembrane transport & 4.1e-81 \\ 
		homeostatic process & 4.8e-103 & \textbf{response to wounding} & 9.4e-80 \\ 
		organophosphate metabolic process & 2.3e-102 & ribonucleotide metabolic process & 3.6e-79 \\ 
		transmembrane transport & 1.5e-99 & purine nucleotide metabolic process & 2.2e-78 \\ 
		response to lipid & 1.2e-97 & organonitrogen compound biosynthetic process & 7.0e-78 \\ 
		response to hormone & 1.5e-97 & anion transport & 4.4e-77 \\ 
		nucleobase-containing small molecule metabolic process & 1.9e-97 & alpha-amino acid metabolic process & 1.1e-76 \\ 
		secretion & 5.4e-97 & purine ribonucleotide metabolic process & 1.3e-76 \\ 
		\textbf{circulatory system process} & 3.4e-96 & inorganic ion transmembrane transport & 1.8e-76 \\ 
		\textbf{blood circulation} & 5.0e-96 & metal ion transport & 2.1e-76 \\ 
		cellular amino acid metabolic process & 4.8e-95 & secretion by cell & 1.7e-73 \\ 
		regulation of ion transport & 2.6e-93 & response to abiotic stimulus & 4.7e-73 \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against GO Biological processes of the list all genes targeted by clinically approved drugs, obtained form DrugBank. For each term, q-values, resulting from Benjamini-Hochberg correction, are reported. Only the first 50 most significant terms are reported.}
	\label{tab:bp2229}
\end{table}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lXlll}
		\rowcolor{NavyBlue!80}
		 \textbf{\color{white} DRUG} & \textbf{\color{white} TARGET} & \textbf{\color{white} DISEASE} & \textbf{\color{white} PAHSE} & \textbf{\color{white} MOLECULE TYPE} \\
		DIPYRIDAMOLE & PDE4B & coronary artery disease & Phase IV & Small molecule \\ 
		INSULIN HUMAN & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		COLLAGENASE CLOSTRIDIUM HISTOLYTICUM & COL5A3 & Dupuytren Contracture & Phase IV & Enzyme \\ 
		METFORMIN & NDUFA4L2 & obesity & Phase IV & Small molecule \\ 
		INSULIN SUSP ISOPHANE BEEF & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		INSULIN LISPRO & INSR & gestational diabetes & Phase IV & Protein \\ 
		INSULIN GLARGINE & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		TRETINOIN & RARA & neoplasm & Phase IV & Small molecule \\ 
		INTERFERON BETA-1B & IFNAR2 & relapsing-remitting multiple sclerosis & Phase IV & Protein \\ 
		INSULIN PORK & INSR & diabetes mellitus & Phase IV & Protein \\ 
		SORAFENIB & FLT1 & hepatocellular carcinoma & Phase IV & Small molecule \\ 
		ADAPALENE & RARA & acne & Phase IV & Small molecule \\ 
		PEGINTERFERON ALFA-2A & IFNAR2 & Chronic Hepatitis C infection & Phase IV & Protein \\ 
		PEGINTERFERON ALFA-2B & IFNAR2 & Chronic Hepatitis C infection & Phase IV & Protein \\ 
		DRONEDARONE & KCNJ2 & atrial fibrillation & Phase IV & Small molecule \\ 
		PENTOXIFYLLINE & PDE4D & Hepatitis, Alcoholic & Phase IV & Small molecule \\ 
		INSULIN PURIFIED BEEF & INSR & diabetes mellitus & Phase IV & Protein \\ 
		INSULIN DETEMIR & INSR & type I diabetes mellitus & Phase IV & Protein \\ 
		AZACITIDINE & DNMT3A & refractory anemia with excess blasts & Phase IV & Small molecule \\ 
		PAZOPANIB & FLT1 & cancer & Phase IV & Small molecule \\ 
		INSULIN GLULISINE & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		THEOPHYLLINE & PDE4B & asthma & Phase IV & Small molecule \\ 
		INSULIN ASPART & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		DECITABINE & DNMT3A & chronic myelomonocytic leukemia & Phase IV & Small molecule \\ 
		DESMOPRESSIN & AVPR1A & Hemophilia A & Phase IV & Protein \\ 
		DYPHYLLINE & PDE4D & obstructive lung disease & Phase IV & Small molecule \\ 
		INTERFERON ALFA-2B & IFNAR2 & mycosis fungoides & Phase IV & Protein \\ 
		INTERFERON BETA-1A & IFNAR2 & multiple sclerosis & Phase IV & Protein \\ 
		ROFLUMILAST & PDE4D & chronic obstructive pulmonary disease & Phase IV & Small molecule \\ 
		ETRETINATE & RARA & psoriasis & Phase IV & Small molecule \\ 
		INSULIN SUSP ISOPHANE RECOMBINANT HUMAN & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		CONIVAPTAN & AVPR1A & cardiovascular disease & Phase IV & Small molecule \\ 
		VASOPRESSIN & AVPR1A & hemorrhage & Phase IV & Protein \\ 
		APREMILAST & PDE4B & Alopecia & Phase IV & Small molecule \\ 
		ALITRETINOIN & RARA & neoplasm & Phase IV & Small molecule \\ 
		ISOTRETINOIN & RARA & acne & Phase IV & Small molecule \\ 
		TAZAROTENE & RARA & acne & Phase IV & Small molecule \\ 
		TIVOZANIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		ACITRETIN & RARA & psoriasis vulgaris & Phase IV & Small molecule \\ 
		LENVATINIB & FLT1 & thyroid carcinoma & Phase IV & Small molecule \\ 
		TOFACITINIB & JAK1 & rheumatoid arthritis & Phase IV & Small molecule \\ 
		PEGINTERFERON BETA-1A & IFNAR2 & relapsing-remitting multiple sclerosis & Phase IV & Protein \\ 
		CRISABOROLE & PDE4B & atopic eczema & Phase IV & Small molecule \\ 
		SUNITINIB & FLT1 & Gastrointestinal stromal tumor & Phase IV & Small molecule \\ 
		INSULIN DEGLUDEC & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		INSULIN LISPRO PROTAMINE RECOMBINANT & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		AXITINIB & FLT1 & renal cell carcinoma & Phase IV & Small molecule \\ 
		NINTEDANIB & FLT1 & neoplasm & Phase IV & Small molecule \\ 
		SILTUXIMAB & IL6 & immune system disease & Phase IV & Antibody \\ 
		AMLEXANOX & PDE4B & obstructive lung disease & Phase IV & Small molecule \\ 
		ANAKINRA & IL1R1 & rheumatoid arthritis & Phase IV & Protein \\ 
		INSULIN SUSP ISOPHANE SEMISYNTHETIC PURIFIED HUMAN & INSR & diabetes mellitus & Phase IV & Protein \\ 
		MIGLITOL & GAA & type II diabetes mellitus & Phase IV & Small molecule \\ 
		REGORAFENIB & FLT1 & colorectal neoplasm & Phase IV & Small molecule \\ 
		VANDETANIB & FLT1 & thyroid carcinoma & Phase IV & Small molecule \\ 
		DROTRECOGIN ALFA (ACTIVATED) & F5 & sepsis & Phase IV & Unknown \\ 
		FIBRINOLYSIN, HUMAN & FGA & Recurrent thrombophlebitis & Phase IV & Unknown \\ 
		COPANLISIB & PIK3CG & neoplasm of mature B-cells & Phase IV & Small molecule \\ 
		FLAVOXATE & PDE4B & pain & Phase IV & Small molecule \\ 
		RUXOLITINIB & JAK1 & polycythemia vera & Phase IV & Small molecule \\ 
		INTERFERON ALFACON-1 & IFNAR2 & hepatitis C infection & Phase IV & Protein \\ 
		INSULIN PURIFIED PORK & INSR & diabetes mellitus & Phase IV & Protein \\ 
		BARICITINIB & JAK1 & immune system disease & Phase IV & Small molecule \\ 
		INSULIN ASPART PROTAMINE RECOMBINANT & INSR & type II diabetes mellitus & Phase IV & Protein \\ 
		INTERFERON ALFA-2A & IFNAR2 & renal cell carcinoma & Phase IV & Protein \\ 
		INSULIN ZINC SUSP RECOMBINANT HUMAN & INSR & diabetes mellitus & Phase IV & Protein \\ 
		OCRIPLASMIN & COL1A1 & macular holes & Phase IV & Enzyme \\ 
		DINOPROSTONE & PTGER4 & pregnancy & Phase IV & Small molecule \\ 
		INTERFERON ALFA-N3 & IFNAR2 & severe acute respiratory syndrome & Phase IV & Protein \\ 
	\end{tabularx}
	\smallskip
	\caption{List of drugs targeting the list of 36 genes resulting from the final step of the pipeline. Only drugs at phase IV of clinical trial are reported.}
	\label{tab:drugs}
\end{table}


\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		response to organic substance & 1.7e-09 & positive regulation of response to external stimulus & 2.1e-03 \\ 
		cellular response to organic ubstance & 3.0e-09 & movement of cell or subcellular component & 2.4e-03 \\ 
		leukocyte migration & 3.8e-08 & intracellular signal transduction & 2.6e-03 \\ 
		circulatory system development & 5.3e-08 & regulation of transport & 2.6e-03 \\ 
		cellular response to chemical stimulus & 1.4e-07 & cellular response to environmental stimulus & 2.7e-03 \\ 
		regulation of multicellular organismal process & 4.6e-07 & cellular response to abiotic stimulus & 2.7e-03 \\ 
		response to chemical & 7.0e-07 & cell communication & 2.7e-03 \\ 
		blood circulation & 1.8e-06 & regulation of phosphorylation & 2.7e-03 \\ 
		multicellular organismal process & 2.0e-06 & cellular response to external stimulus & 2.8e-03 \\ 
		circulatory system process & 2.2e-06 & regulation of cytokine production & 2.9e-03 \\ 
		myeloid leukocyte migration & 2.5e-06 & cellular response to biotic stimulus & 2.9e-03 \\ 
		cellular response to oxygen-containing compound & 3.2e-06 & interferon-gamma production & 3.0e-03 \\ 
		cellular response to organonitrogen compound & 4.2e-06 & system process & 3.1e-03 \\ 
		response to external stimulus & 4.4e-06 & cellular developmental process & 3.2e-03 \\ 
		cellular response to endogenous stimulus & 8.3e-06 & response to bacterium & 3.4e-03 \\ 
		response to endogenous stimulus & 8.4e-06 & cell proliferation & 3.4e-03 \\ 
		response to organonitrogen compound & 1.1e-05 & cellular response to cytokine stimulus & 3.6e-03 \\ 
		immune response & 1.1e-05 & regulation of cell adhesion & 3.7e-03 \\ 
		cellular response to mechanical stimulus & 1.3e-05 & cellular response to acid chemical & 3.8e-03 \\ 
		protein phosphorylation & 1.6e-05 & leukocyte chemotaxis & 3.8e-03 \\ 
		cellular response to nitrogen compound & 2.3e-05 & collagen-activated signaling pathway & 3.9e-03 \\ 
		cell surface receptor signaling pathway & 3.3e-05 & secretion & 4.2e-03 \\ 
		regulation of response to cytokine stimulus & 3.6e-05 & response to mechanical stimulus & 5.0e-03 \\ 
		response to nitrogen compound & 4.7e-05 & regulation of heart contraction & 5.3e-03 \\ 
		extracellular structure organization & 4.9e-05 & positive regulation of biological process & 5.5e-03 \\ 
		response to stimulus & 5.4e-05 & blood coagulation & 5.6e-03 \\ 
		immune system process & 5.5e-05 & positive regulation of cell migration & 5.6e-03 \\ 
		response to oxygen-containing compound & 6.7e-05 & regulation of secretion by cell & 6.1e-03 \\ 
		multi-organism process & 6.9e-05 & hemostasis & 6.1e-03 \\ 
		regulation of cell communication & 7.0e-05 & positive regulation of nitric-oxide synthase biosynthetic process & 6.2e-03 \\ 
		tube morphogenesis & 7.1e-05 & coagulation & 6.4e-03 \\ 
		regulation of signaling & 8.4e-05 & positive regulation of cell motility & 7.2e-03 \\ 
		wound healing & 8.6e-05 & cell activation & 7.4e-03 \\ 
		response to cytokine & 9.8e-05 & defense response & 7.8e-03 \\ 
		blood vessel development & 1.1e-04 & positive regulation of inflammatory response & 7.9e-03 \\ 
		response to organic cyclic compound & 1.3e-04 & negative regulation of relaxation of cardiac muscle & 8.2e-03 \\ 
		positive regulation of multicellular organismal process & 1.4e-04 & positive regulation of secretion by cell & 8.3e-03 \\ 
		cellular response to stimulus & 1.5e-04 & positive regulation of cellular component movement & 9.0e-03 \\ 
		regulation of immune response & 1.7e-04 & localization & 9.1e-03 \\ 
		anatomical structure development & 1.7e-04 & relaxation of cardiac muscle & 9.3e-03 \\ 
		vasculature development & 1.8e-04 & response to other organism & 9.9e-03 \\ 
		response to lipopolysaccharide & 1.9e-04 & response to external biotic stimulus & 1.0e-02 \\ 
		cardiovascular system development & 2.0e-04 & regulation of phosphate metabolic process & 1.1e-02 \\ 
		granulocyte migration & 2.0e-04 & regulation of phosphorus metabolic process & 1.1e-02 \\ 
		negative regulation of multicellular organismal process & 2.1e-04 & positive regulation of cell adhesion & 1.1e-02 \\ 
		regulation of cellular component movement & 2.3e-04 & regulation of secretion & 1.2e-02 \\ 
		cell migration & 2.4e-04 & regulation of defense response & 1.2e-02 \\ 
		extracellular matrix organization & 2.7e-04 & regulation of system process & 1.2e-02 \\ 
		phosphorylation & 2.7e-04 & peptidyl-tyrosine phosphorylation & 1.3e-02 \\ 
		regulation of response to stimulus & 2.8e-04 & positive regulation of interferon-gamma production & 1.3e-02 \\ 
		response to molecule of bacterial origin & 2.9e-04 & negative regulation of cytokine production & 1.3e-02 \\ 
		regulation of localization & 3.2e-04 & regulation of MAPK cascade & 1.3e-02 \\ 
		regulation of immune system process & 3.2e-04 & nitric-oxide synthase biosynthetic process & 1.3e-02 \\ 
		cardiac muscle contraction & 3.4e-04 & regulation of nitric-oxide synthase biosynthetic process & 1.3e-02 \\ 
		signaling & 3.8e-04 & peptidyl-tyrosine modification & 1.3e-02 \\ 
		signal transduction & 4.2e-04 & positive regulation of protein phosphorylation & 1.3e-02 \\ 
		blood vessel morphogenesis & 4.4e-04 & positive regulation of secretion & 1.4e-02 \\ 
		response to wounding & 4.6e-04 & cytokine-mediated signaling pathway & 1.4e-02 \\ 
		tube development & 5.2e-04 & immune effector process & 1.5e-02 \\ 
		locomotion & 5.5e-04 & positive regulation of immune response & 1.5e-02 \\ 
		heart contraction & 6.0e-04 & response to biotic stimulus & 1.5e-02 \\ 
		regulation of signal transduction & 6.4e-04 & regulation of locomotion & 1.6e-02 \\ 
		cytokine production & 6.5e-04 & cellular response to lipid & 1.6e-02 \\ 
		MAPK cascade & 6.8e-04 & macromolecule modification & 1.8e-02 \\ 
		system development & 7.3e-04 & chemotaxis & 1.8e-02 \\
		heart process & 7.4e-04 & taxis & 1.8e-02 \\	
		localization of cell & 7.5e-04 & regulation of cytokine production involved in immune response & 1.8e-02 \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against the GO Biological Processes of the list of 36 genes, resulting from the final step of the pipeline. For each term, q-values, resulting from Benjamini-Hochberg correction, are reported. (Table continued in Tab. \ref{tab:GO36_2})}
	\label{tab:GO36_1}
\end{table}

			
\begin{table}[!htb]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		cell motility & 7.5e-04 & regulation of protein modification process & 1.9e-02 \\ 
		regulation of cytokine-mediated signaling pathway & 7.5e-04 & positive regulation of phosphorylation & 2.0e-02 \\ 
		platelet activation & 7.8e-04 & enzyme linked receptor protein signaling pathway & 2.4e-02 \\ 
		transmembrane receptor protein tyrosine kinase signaling pathway & 8.1e-04 & regulation of response to external stimulus & 2.4e-02 \\ 
		developmental process & 8.2e-04 & positive regulation of intracellular signal transduction & 2.4e-02 \\ 
		signal transduction by protein phosphorylation & 8.3e-04 & positive regulation of response to macrophage colony-stimulating factor & 2.5e-02 \\ 
		response to lipid & 8.4e-04 & positive regulation of cellular response to macrophage colony-stimulating factor stimulus & 2.5e-02 \\ 
		positive regulation of locomotion & 9.3e-04 & positive regulation of signal transduction & 2.6e-02 \\ 
		regulation of blood circulation & 9.6e-04 & positive regulation of immune system process & 2.8e-02 \\ 
		anatomical structure formation involved in morphogenesis & 9.8e-04 & regulation of intracellular signal transduction & 2.9e-02 \\ 
		phosphate-containing compound metabolic process & 1.0e-03 & mononuclear cell migration & 3.3e-02 \\ 
		phosphorus metabolic process & 1.1e-03 & regulation of cardiac muscle contraction & 3.3e-02 \\ 
		multicellular organism development & 1.1e-03 & lymphocyte activation involved in immune response & 3.4e-02 \\ 
		cell chemotaxis & 1.1e-03 & positive regulation of phosphorus metabolic process & 3.5e-02 \\ 
		positive regulation of cell proliferation & 1.1e-03 & positive regulation of phosphate metabolic process & 3.5e-02 \\ 
		cellular response to lipopolysaccharide & 1.1e-03 & regulation of biological quality & 3.5e-02 \\ 
		positive regulation of cytokine production & 1.2e-03 & tissue development & 3.9e-02 \\ 
		regulation of protein phosphorylation & 1.3e-03 & regulation of cell migration & 4.0e-02 \\ 
		anatomical structure morphogenesis & 1.4e-03 & protein modification process & 4.1e-02 \\ 
		angiogenesis & 1.5e-03 & cellular protein modification process & 4.1e-02 \\ 
		positive regulation of response to stimulus & 1.5e-03 & transport & 4.2e-02 \\ 
		cell differentiation & 1.5e-03 & muscle system process & 4.2e-02 \\ 
		cellular response to molecule of bacterial origin & 1.5e-03 & positive regulation of production of molecular mediator of immune response & 4.2e-02 \\ 
		secretion by cell & 1.5e-03 & regulation of leukocyte migration & 4.2e-02 \\ 
		regulation of cell proliferation & 1.6e-03 & positive regulation of metabolic process & 4.6e-02 \\ 
		collagen-activated tyrosine kinase receptor signaling pathway & 1.7e-03 & cytokine production involved in immune response & 4.6e-02 \\ 
		regulation of interferon-gamma production & 1.7e-03 & positive regulation of developmental process & 4.8e-02 \\ 
		striated muscle contraction & 1.9e-03 & positive regulation of immune effector process & 4.8e-02 \\ 
		inflammatory response & 2.0e-03 &  &  \\
	\end{tabularx}
	\smallskip
	\caption{Continued from Tab. \ref{tab:GO36_1}}
	\label{tab:GO36_2}
	\bigskip\bigskip
	\centering
	\scriptsize
	\rowcolors{2}{NavyBlue!10}{white}
	\begin{tabularx}{\textwidth}{XlXl}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} GO term} & \textbf{\color{white} q-value} & \textbf{\color{white} GO term} & \textbf{\color{white} q-value} \\
		cellular response to organic substance & 5.1e-07 & response to interleukin-6 & 9.1e-03 \\ 
		cellular response to chemical stimulus & 5.8e-06 & signaling & 9.2e-03 \\ 
		response to organic substance & 7.6e-06 & cardiac muscle contraction & 9.5e-03 \\ 
		response to organonitrogen compound & 4.9e-05 & regulation of system process & 9.6e-03 \\ 
		regulation of multicellular organismal process & 5.7e-05 & response to external stimulus & 1.0e-02 \\ 
		cellular response to oxygen-containing compound & 1.2e-04 & regulation of transport & 1.1e-02 \\ 
		leukocyte migration & 1.3e-04 & regulation of cardiac muscle cell contraction & 1.2e-02 \\ 
		collagen-activated tyrosine kinase receptor signaling pathway & 1.3e-04 & extracellular matrix organization & 1.4e-02 \\ 
		response to nitrogen compound & 1.4e-04 & response to organic cyclic compound & 1.6e-02 \\ 
		circulatory system development & 2.2e-04 & regulation of actin filament-based movement & 1.6e-02 \\ 
		response to endogenous stimulus & 2.8e-04 & platelet activation & 1.6e-02 \\ 
		collagen-activated signaling pathway & 3.1e-04 & regulation of immune response & 1.9e-02 \\ 
		cellular response to organonitrogen compound & 3.8e-04 & acute-phase response & 2.0e-02 \\ 
		relaxation of cardiac muscle & 7.4e-04 & cellular response to lipopolysaccharide & 2.1e-02 \\ 
		cell surface receptor signaling pathway & 8.4e-04 & negative regulation of relaxation of muscle & 2.1e-02 \\ 
		response to chemical & 9.4e-04 & cellular response to molecule of bacterial origin & 2.5e-02 \\ 
		cellular response to endogenous stimulus & 1.0e-03 & cell migration & 2.6e-02 \\ 
		cellular response to nitrogen compound & 1.2e-03 & striated muscle contraction & 3.0e-02 \\ 
		negative regulation of relaxation of cardiac muscle & 1.4e-03 & extracellular structure organization & 3.0e-02 \\ 
		regulation of cellular component movement & 1.5e-03 & regulation of relaxation of cardiac muscle & 3.0e-02 \\ 
		heart contraction & 3.3e-03 & regulation of interleukin-1-mediated signaling pathway & 3.0e-02 \\ 
		regulation of localization & 3.5e-03 & myeloid leukocyte migration & 3.2e-02 \\ 
		cellular response to stimulus & 3.7e-03 & cellular response to cytokine stimulus & 3.2e-02 \\ 
		response to oxygen-containing compound & 3.8e-03 & signal transduction & 3.6e-02 \\ 
		heart process & 3.8e-03 & movement of cell or subcellular component & 3.6e-02 \\ 
		regulation of blood circulation & 4.7e-03 & cellular response to biotic stimulus & 4.0e-02 \\ 
		blood circulation & 5.4e-03 & system process & 4.2e-02 \\ 
		positive regulation of multicellular organismal process & 5.9e-03 & positive regulation of interferon-gamma production & 4.2e-02 \\ 
		relaxation of muscle & 5.9e-03 & regulation of cell communication & 4.5e-02 \\ 
		circulatory system process & 6.0e-03 & cellular response to acid chemical & 4.8e-02 \\ 
		negative regulation of multicellular organismal process & 6.4e-03 & regulation of signaling & 5.0e-02 \\ 
		cellular response to interleukin-6 & 6.5e-03 &  &  \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against the GO Biological Processes of the list of 18 genes, resulting from the final step of the pipeline. For each term, q-values, resulting from Benjamini-Hochberg correction, are reported.}
	\label{tab:GO18}
\end{table}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Disease} & \textbf{\color{white} q-value} & \textbf{\color{white} Disease} & \textbf{\color{white} q-value} \\
		Endometriosis & 3.14e-10 & Epithelial ovarian cancer & 4.74e-07 \\ 
		Myocardial Infarction & 1.36e-09 & Nephrotic Syndrome & 7.05e-07 \\ 
		Rheumatoid Arthritis & 8.56e-09 & Malignant tumor of cervix & 7.09e-07 \\ 
		Atherosclerosis & 5.34e-08 & Cirrhosis & 8.09e-07 \\ 
		Scleroderma & 5.34e-08 & Ear, patella, short stature syndrome & 8.19e-07 \\ 
		Lymphoma & 5.34e-08 & Cerebrovascular accident & 9.14e-07 \\ 
		Diabetes Mellitus & 5.34e-08 & Fibrosarcoma & 9.29e-07 \\ 
		Arteriosclerosis & 5.34e-08 & Epithelioma & 9.29e-07 \\ 
		Cardiovascular Diseases & 5.52e-08 & Exanthema & 1.18e-06 \\ 
		Diabetes & 5.77e-08 & Cervical cancer & 1.20e-06 \\ 
		Hypertensive disease & 7.17e-08 & Mammary Neoplasms & 1.37e-06 \\ 
		Systemic Scleroderma & 1.13e-07 & Fibrosis, Liver & 1.40e-06 \\ 
		Obesity & 1.13e-07 & Kidney Diseases & 1.40e-06 \\ 
		Diabetic Nephropathy & 1.13e-07 & Colitis & 1.40e-06 \\ 
		Leukemia, Myelocytic, Acute & 2.09e-07 & Liver diseases & 1.55e-06 \\ 
		Elastosis perforans serpiginosa & 2.21e-07 & Hamartoma Syndrome, Multiple & 1.72e-06 \\ 
		Malignant neoplasm of ovary & 2.39e-07 & Hyperglycemia & 1.79e-06 \\ 
		Acute lymphocytic leukemia & 2.90e-07 & Amyloidosis & 1.84e-06 \\ 
		Degenerative polyarthritis & 3.22e-07 & Pancreatic carcinoma & 2.00e-06 \\ 
		Pain & 3.92e-07 & Chronic myeloproliferative disorder & 2.01e-06 \\ 
		Liver neoplasms & 4.53e-07 & Overweight & 2.17e-06 \\ 
		Kidney Failure & 4.53e-07 & Acute Promyelocytic Leukemia & 2.32e-06 \\ 
		Psoriasis & 4.66e-07 & Esophageal carcinoma & 2.51e-06 \\ 
		Liver Cirrhosis & 4.66e-07 & Lupus Erythematosus, Systemic & 2.51e-06 \\ 
		Acute leukemia & 4.73e-07 & Hamman-Rich syndrome & 2.62e-06 \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against diseases databases of the list of 36 genes, resulting from the final step of the pipeline. For each disease, q-values, resulting from Benjamini-Hochberg correction, are reported. Only the first 50 most significant diseases are reported.}
	\label{tab:diseases36}
\end{table}

\begin{table}[ht]
	\centering
	\scriptsize
	\rowcolors{2}{white}{NavyBlue!10}
	\begin{tabularx}{\textwidth}{lRlR}
		\rowcolor{NavyBlue!80}
		\textbf{\color{white} Disease} & \textbf{\color{white} q-value} & \textbf{\color{white} Disease} & \textbf{\color{white} q-value} \\
		Keloid & 3.98e-05 & Aortic Valve Insufficiency & 5.00e-04 \\ 
		Obesity & 4.10e-05 & Anemia of chronic disease & 5.00e-04 \\ 
		Diffuse Scleroderma & 7.65e-05 & Ear, patella, short stature syndrome & 5.31e-04 \\ 
		Fibrosis, Liver & 1.40e-04 & Spontaneous abortion & 5.49e-04 \\ 
		Joint laxity & 1.99e-04 & Acute Cholecystitis & 5.62e-04 \\ 
		Cardiovascular Diseases & 1.99e-04 & Atherosclerosis & 5.76e-04 \\ 
		Sick Sinus Syndrome & 1.99e-04 & Kidney Failure & 5.76e-04 \\ 
		Myocardial Infarction & 1.99e-04 & Inflammatory Myofibroblastic Tumor & 5.76e-04 \\ 
		Diabetes Mellitus & 1.99e-04 & Arteriosclerosis & 5.76e-04 \\ 
		Hypertensive disease & 2.18e-04 & Nephrotic Syndrome & 5.76e-04 \\ 
		Porencephaly, familial & 2.45e-04 & Cerebral Autosomal Recessive Arteriopathy with Subcortical Infarcts and Leukoencephalopathy & 7.80e-04 \\ 
		Scleroderma & 2.45e-04 & Refractory anemias & 8.06e-04 \\ 
		Kidney Diseases & 2.68e-04 & Focal glomerulosclerosis & 8.43e-04 \\ 
		Chronic glomerulonephritis & 3.02e-04 & Diabetic Nephropathy & 9.37e-04 \\ 
		Elastosis perforans serpiginosa & 3.24e-04 & Rheumatoid Arthritis & 1.37e-03 \\ 
		Hereditary gingival fibromatosis & 4.05e-04 & Ehlers-Danlos Syndrome & 1.45e-03 \\ 
		Diabetes & 4.15e-04 & Hamman-Rich syndrome & 1.45e-03 \\ 
		Alcoholic Intoxication, Chronic & 4.34e-04 & Lymphoma & 1.48e-03 \\ 
		Hemorrhage, intracerebral, susceptibility to & 4.44e-04 & Subcutaneous nodule & 1.48e-03 \\ 
		Parkinson disease, late-onset & 4.54e-04 & Multiple, subcutaneous nodules & 1.48e-03 \\ 
		Endometriosis & 4.59e-04 & Uterine Fibroids & 1.48e-03 \\ 
		Atrial Fibrillation & 4.66e-04 & Nonalcoholic Steatohepatitis & 1.56e-03 \\ 
		Renal Insufficiency & 4.84e-04 & Chronic lung disease & 1.65e-03 \\ 
		Overweight & 4.84e-04 & Retinal Detachment & 1.65e-03 \\ 
		Lymphoproliferative Disorder of the Skin & 4.90e-04 & Gastroesophageal reflux disease & 1.65e-03 \\ 
	\end{tabularx}
	\smallskip
	\caption{Result of the enrichment analysis against diseases databases of the list of 18 genes, resulting from the final step of the pipeline. For each disease, q-values, resulting from Benjamini-Hochberg correction, are reported. Only the first 50 most significant diseases are reported.}
	\label{tab:diseases18}
\end{table}

\end{document}