%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Stylish Article
% LaTeX Template
% Version 2.1 (1/10/15)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Mathias Legrand (legrand.mathias@gmail.com) 
% With extensive modifications by:
% Vel (vel@latextemplates.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[fleqn,10pt]{SelfArx} % Document font size and equations flushed left

\usepackage[english]{babel} % Specify a different language here - english by default

\usepackage{lipsum} % Required to insert dummy text. To be removed otherwise

%----------------------------------------------------------------------------------------
%	COLUMNS
%----------------------------------------------------------------------------------------

\setlength{\columnsep}{0.55cm} % Distance between the two columns of text
\setlength{\fboxrule}{0.75pt} % Width of the border around the abstract

%----------------------------------------------------------------------------------------
%	COLORS
%----------------------------------------------------------------------------------------

\definecolor{color1}{RGB}{0,0,100} % Color of the article title and sections
\definecolor{color2}{RGB}{0,10,10} % Color of the boxes behind the abstract and headings

%----------------------------------------------------------------------------------------
%	HYPERLINKS
%----------------------------------------------------------------------------------------

\usepackage{hyperref} % Required for hyperlinks
\hypersetup{hidelinks,colorlinks,breaklinks=true,urlcolor=color2,citecolor=color1,linkcolor=color1,bookmarksopen=false,pdftitle={Title},pdfauthor={Author}}

%----------------------------------------------------------------------------------------
%	ARTICLE INFORMATION
%----------------------------------------------------------------------------------------

\JournalInfo{Laboratory of Data Mining Report} % Journal information
\Archive{} % Additional notes (e.g. copyright, DOI, review/research article)

\PaperTitle{Article Title} % Article title

\Authors{John Smith\textsuperscript{1}*, James Smith\textsuperscript{2}} % Authors
\affiliation{\textsuperscript{1}\textit{Department of Biology, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{\textsuperscript{2}\textit{Department of Chemistry, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{*\textbf{Corresponding author}: john@smith.com} % Corresponding author

\Keywords{Keyword1 --- Keyword2 --- Keyword3} % Keywords - if you don't want any simply remove all the text between the curly brackets
\newcommand{\keywordname}{Keywords} % Defines the keywords heading name

%----------------------------------------------------------------------------------------
%	ABSTRACT
%----------------------------------------------------------------------------------------

\Abstract{\lipsum[1]~}

%----------------------------------------------------------------------------------------

\begin{document}

\flushbottom % Makes all text pages the same height

\maketitle % Print the title and abstract box

\tableofcontents % Print the contents section

\thispagestyle{empty} % Removes page numbering from the first page

%----------------------------------------------------------------------------------------
%	ARTICLE CONTENTS
%----------------------------------------------------------------------------------------

\section*{Introduction} % The \section*{} command stops section numbering

\addcontentsline{toc}{section}{Introduction} % Adds this section to the table of contents

Ele

general intro to drug repurposing

intro to cad disease

intro to nessra

basic idea of the pipeline

\section*{Materials and Methods}

\addcontentsline{toc}{section}{Materials and Methods} % Adds this section to the table of contents

\subsection*{Data}

\subsection*{Platforms and Datasources}
Nessra is designed in three steps in which first subsets data,
secondly runs the skeleton of the PC algorithm and finally aggregates results, computing the frequencies of the transcripts.

For the most computationally demanding step of Nessra, which consists in running the skeleton of the PC algorithm, the sigle gene expansions were run within the local gene@home \cite{boinc} BOINC (Berkeley Open Infrastructure for Network Computing) project, hosted by the TN-Grid platform. 
The platform allowed to distribute the expansions computational burden to committed volunteers' machines in order to reduce the running time and memory usage of the algorithm as much as possible.
This gene@home project was previously set up for the network expansion of genes whose expression data came from the FANTOM5 Database \cite{fantom}, so this guided the choice of the transcriptional data to run Nessra on.
To adress the drug repurposing task, Open Targets\cite{open} platform was chosen to select genes involved in the CAD disease.  


\subsection*{Pipeline}

\addcontentsline{toc}{subsection}{Pipeline} % Adds this section to the table of contents

Enri

selection of genes to expand

nessra expansion of cad associated genes

score aggregation of each isoform

filter on isoforms

Matteo

The resulting n isoforms were ranked according to the aggregated score described previously. Then, the ranked list of isoforms were was converted in a ranked list of genes. The rank of a gene was obtained as the minimum of the ranks of its isoforms.

The OpenTragets pyhton API was used to query which of these genes are known to be targets of clinically approved drugs. We obtained a list of 148 target genes.

In order to identify a subset of genes with a strong association with CAD, we selected the isoforms corresponding to these 148 target genes and we performed single isoform expansion using NES$^2$RA. Only the isoforms that passed the previous filter were considered. We expect NES$^2$RA expansion of genes that have strong interaction with CAD genes to have a large overlap with the list of CAD genes we started form.

In order to quantify the overlap between the ranked list of CAD genes and the ranked expansion lists obtained with NES$^2$RA, we used the weighted Jaccard similarity (WJS), defined as follows:

\textbf{Definition}: given two weighted list of features, $\rho$ and $\sigma$, of equal length $N$, their weighted Jaccard similarity, $WJS(\rho, \sigma)$ is defined as:

$$
WJS(\rho, \sigma) = \dfrac{\sum_{i=1}^Nmin(\rho_i,\sigma_i)}{\sum_{i=1}^Nmax(\rho_i,\sigma_i)}
$$

where $\rho_i$ and $\sigma_i$ are the weights corresponding to the same feature $i$.

In our analysis the weight of a feature $i$ (gene or isoform) in a ranked list $\rho$ is computed as $length(\rho) - rank_{\rho}(i) + 1$. This allows to assign large weights to high ranking features.

Since in principle the lists of features that we want to compare with the WJS do not contain the same elements, we need to add the missing elements to those lists. Given two lists of features $\rho$ and $\sigma$, containing different elements, we added the missing elements of one list to the other and viceversa. The added elements are added as ties in the last position of the ranking.

In order to compute the WJS of the ranked list of CAD genes and a ranked list of isoforms obtained using NES$^2$RA, we have to either convert the ranked list of CAD genes in a ranked list of isoforms or convert the ranked  list of isoforms obtained using NES$^2$RA in a ranked list of genes. We explored both options.

\subsubsection{Convert NES$^2$RA isoforms into genes}

Given list of isoforms obteined with NES$^2$RA, we can convert it in a list of genes, assigning to each gene the largest relative frequency of its isoforms. Then the genes were ranked according to relative frequency.

\subsubsection{Convert CAD genes into isoforms}

The ranked list of CAD genes was converted to a weighted list of isoforms, were the weight of each isoform corresponded to the rank of the genes. Then the list of isoforms was ranked according to the weights. This way, isoforms corresponding to the same gene presented the same rank.

We computed the WJS of the ranked list of CAD genes (respectively, isoforms) and the ranked lists of genes (respectively, isoforms) obtained with NES$^2$RA. 

The scores obtained in different comparisons are not directly comparable. This is due to the different length of the lists. In order to obtain values that can be compared directly, we used a permutation approach to estimate the distribution of scores of lists of different length. For each length present, we generated 2000 random lists of genes or isoforms, we computed the WJS and we generated a distribution of the scores. Then, we used this distribution to compute p-values.

...

enrichment analysis

\section*{Results}

Matteo

tables of genes found

results of the enrichment analysis

\section*{Discussion}

Chiara

gene that are already targets of drugs used for cad

interpretation of the enrichment analysis

discussion of new targets and drugs

\section*{Conclusions}

necessity of experimental validations

\phantomsection
%----------------------------------------------------------------------------------------
%	REFERENCE LIST
%----------------------------------------------------------------------------------------
\phantomsection
\bibliographystyle{unsrt}
\bibliography{biblio}
\begin{thebibliography}{1}
	
	\bibitem{boinc} Asnicar, F. et al. NES2RA. Int. J. High Perform. Comput. Appl. 32, 380â€“392 (2016).
	
	
	\bibitem{fantom} Noguchi, S. et al. FANTOM5 CAGE profiles of human and mouse samples. Sci Data 4, 170112 (2017).
	
	
	\bibitem{open} Koscielny, G. et al. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 45, D985â€“D994 (2017).
	
	
\end{thebibliography}

%----------------------------------------------------------------------------------------

\end{document}