%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Stylish Article
% LaTeX Template
% Version 2.1 (1/10/15)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Mathias Legrand (legrand.mathias@gmail.com) 
% With extensive modifications by:
% Vel (vel@latextemplates.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[fleqn,10pt]{SelfArx} % Document font size and equations flushed left

\usepackage[english]{babel} % Specify a different language here - english by default

\usepackage{lipsum} % Required to insert dummy text. To be removed otherwise

%----------------------------------------------------------------------------------------
%	COLUMNS
%----------------------------------------------------------------------------------------

\setlength{\columnsep}{0.55cm} % Distance between the two columns of text
\setlength{\fboxrule}{0.75pt} % Width of the border around the abstract

%----------------------------------------------------------------------------------------
%	COLORS
%----------------------------------------------------------------------------------------

\definecolor{color1}{RGB}{0,0,100} % Color of the article title and sections
\definecolor{color2}{RGB}{0,10,10} % Color of the boxes behind the abstract and headings

%----------------------------------------------------------------------------------------
%	HYPERLINKS
%----------------------------------------------------------------------------------------

\usepackage{hyperref} % Required for hyperlinks
\hypersetup{hidelinks,colorlinks,breaklinks=true,urlcolor=color2,citecolor=color1,linkcolor=color1,bookmarksopen=false,pdftitle={Title},pdfauthor={Author}}

%----------------------------------------------------------------------------------------
%	ARTICLE INFORMATION
%----------------------------------------------------------------------------------------

\JournalInfo{Laboratory of Data Mining Report} % Journal information
\Archive{} % Additional notes (e.g. copyright, DOI, review/research article)

\PaperTitle{Article Title} % Article title

\Authors{John Smith\textsuperscript{1}*, James Smith\textsuperscript{2}} % Authors
\affiliation{\textsuperscript{1}\textit{Department of Biology, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{\textsuperscript{2}\textit{Department of Chemistry, University of Examples, London, United Kingdom}} % Author affiliation
\affiliation{*\textbf{Corresponding author}: john@smith.com} % Corresponding author

\Keywords{Keyword1 --- Keyword2 --- Keyword3} % Keywords - if you don't want any simply remove all the text between the curly brackets
\newcommand{\keywordname}{Keywords} % Defines the keywords heading name

%----------------------------------------------------------------------------------------
%	ABSTRACT
%----------------------------------------------------------------------------------------

\Abstract{\lipsum[1]~}

%----------------------------------------------------------------------------------------

\begin{document}

\flushbottom % Makes all text pages the same height

\maketitle % Print the title and abstract box

\tableofcontents % Print the contents section

\thispagestyle{empty} % Removes page numbering from the first page

%----------------------------------------------------------------------------------------
%	ARTICLE CONTENTS
%----------------------------------------------------------------------------------------

\section*{Introduction} % The \section*{} command stops section numbering

\addcontentsline{toc}{section}{Introduction} % Adds this section to the table of contents

Drug repurposing is a strategy to identify new therapeutics uses for marketed drugs \cite{Polamreddy}. This procedure has known for time but only in recent years its use has increased due to new molecular discoveries, new high-throughput technologies and databases. In average the launch of a new drug on the market takes 14 years, drug repurposing can drastically decrease this time because of the usage of previous knowledge and, moreover, it can lower the costs of the R{\&}D process.\\
\\
Coronary artery disease (CAD) is the most common type of heart disease.It is the leading cause of death in the United States and in Italy (WHO data). The condition leads the formation of a waxy substance calle plaque into coronary arteries decreasing the flow of oxygen-rich blood to the heart. The condition could worsen after the plaque rupture due to the formation of blood clot that could mostrly (or completely) block blood flow through a coronary artery. The disease can weaken the heart muscle and lead to heart failure and arrhytmias, moreover it cas cause angina and heart attack.\\
\\
In this project we apply NES2RA\cite{NES2RA}, a pipeline that finds candidate genes to expand a known gene-network based PC-algorithm \cite{PC-alg}, to find marketed drug that can be repurposed for CAD. 
The pipeline of our work can be divided in 2 significant steps: retrieving known target genes, backward expansion and analysis.\\
\\
In the first step the ranked CAD genetic associated genes are obtained using Open Target \cite{OpenTargetPlatform}, we decide to focus on the 100 most important ones without minding if there are already known drug target for the disease.\\
For each gene the corresponding isoforms are picked from the filtered annotation of the FANTOM5 database \cite{fantom} and then expanded by NES2RA using the Boinc Platform \cite{realBoinc} and FANTOM5. NES2RA gives an expansion file for each isoform, where isoforms related with the one in input are collected and ranked by an association score. Minding the importance of the previous 100 CAD genes and the overall score for each isoform , the expansion files are filtered such that a list of particularly interesting isoforms is obtained. Successively using the Open Target Platform, we extract the gens that are already known targets for any drug from the aforementioned list. At the end of this step we are left with a list of 148 drugged genes corresponding to 217 isoforms.\\
\\
The purpose of the second step is to verify which of the genes has a drug that can be repurposed for CAD.
Firstly a backward expansion on the 217 isoforms is performed using NES2RA.
Secondly an analsyis using weighted Jaccard Similarity (WGS) \cite{WGS} for each expansion result paired with the initial CAD genetic associated genes is done to obtain a p-value for possible relation among the isoform in input and the CAD. 

The aim of the project is to see if NES2RA can be proposed as a new \textit{in silico} tool for drug repurposing and to find possible new drugs for the CAD disease.  


\section*{Materials and Methods}
\addcontentsline{toc}{section}{Materials and Methods} % Adds this section to the table of contents


\addcontentsline{toc}{section}{Materials and Methods} % Adds this section to the table of contents

\subsection*{Platforms and Datasources}
\addcontentsline{toc}{subsection}{Platforms and Datasources} % Adds this section to the table of contents

Nessra is designed in three steps in which first subsets data,
secondly runs the skeleton of the PC algorithm and finally aggregates results, computing the frequencies of the transcripts.

For the most computationally demanding step of Nessra, which consists in running the skeleton of the PC algorithm, the sigle gene expansions were run within the local gene@home \cite{boinc} BOINC (Berkeley Open Infrastructure for Network Computing) project, hosted by the TN-Grid platform. 
The platform allowed to distribute the expansions computational burden to committed volunteers' machines in order to reduce the running time and memory usage of the algorithm as much as possible.
This gene@home project was previously set up for the network expansion of genes whose expression data came from the FANTOM5 Database \cite{fantom}, so this guided the choice of the transcriptional data to run Nessra on.
To adress the drug repurposing task, Open Targets\cite{open} platform was chosen to select genes involved in the CAD disease.  


\subsection*{Pipeline}
<<<<<<< HEAD
\addcontentsline{toc}{subsection}{Pipeline} % Adds this section to the table of contents
=======


>>>>>>> e0690675f635b946caf5403f9a7f9ce7615e2a79
Enri

selection of genes to expand

nessra expansion of cad associated genes

score aggregation of each isoform

filter on isoforms

Matteo

The resulting n isoforms were ranked according to the aggregated score described previously. Then, the ranked list of isoforms was converted in a ranked list of genes. The rank of a gene was obtained as the minimum of the ranks of its isoforms.

\subsubsection{Selection of target genes}

The Open Tragets pyhton API [reference?] was used to query which of these genes are known to be targets of clinically approved drugs. We obtained a list of 148 target genes, which are the only ones that are considered in the following analyses.

\subsubsection{NES$^{\textbf{2}}$RA expansion of target genes}

In order to identify a subset of genes with a strong association with CAD, we selected the isoforms corresponding to these 148 target genes and we performed single isoform expansions using NES$^2$RA. Only the isoforms that passed the previous filter were considered. We expect NES$^2$RA expansions of genes that have strong interaction with CAD genes to have a large overlap with the list of CAD genes we started form.

\subsubsection{Comparison of expanded lists to CAD genes}

In order to quantify the overlap between the ranked list of CAD genes and the ranked expansion lists previously obtained, we used the weighted Jaccard similarity (WJS), which we define below.\medskip

\noindent
\textit{Definition: given two weighted list of items, $\rho$ and $\sigma$, of equal length $N$, their weighted Jaccard similarity, $WJS(\rho, \sigma)$, is defined as:}

$$
WJS(\rho, \sigma) = \dfrac{\sum_{i=1}^Nmin(\rho_i,\sigma_i)}{\sum_{i=1}^Nmax(\rho_i,\sigma_i)}
$$

\noindent
\textit{where $\rho_i$ and $\sigma_i$ are the weights corresponding to the same feature $i$.}\medskip

In our analysis the weight of a feature $i$ (gene or isoform) in a ranked list $\rho$ is computed as $length(\rho) - rank_{\rho}(i) + 1$. This allows to assign large weights to high ranking features.

Since in principle the lists of features that we want to compare with the WJS do not contain the same elements, we need to add the missing elements to those lists. Given two lists of features $\rho$ and $\sigma$, containing different elements, we added the missing elements of one list to the other and viceversa. The missing elements are added as ties in the last position of the ranking.

In order to compute the WJS of the ranked list of CAD genes and a ranked list of isoforms obtained using NES$^2$RA, we have to either convert the ranked list of CAD genes in a ranked list of isoforms or convert the ranked list of isoforms obtained using NES$^2$RA in a ranked list of genes. We explored both options:

\begin{itemize}
	\item \textit{Convert NES$^2$RA isoforms into genes}: Given list of isoforms obtained with NES$^2$RA, we can convert it in a list of genes, assigning to each gene the largest relative frequency of its isoforms. Then the genes were ranked according to relative frequency.
	\item \textit{Convert CAD genes into isoforms}: The ranked list of CAD genes was converted to a weighted list of isoforms, were the weight of each isoform corresponded to the rank of the genes. Then the list of isoforms was ranked according to the weights. This way, isoforms corresponding to the same gene presented the same rank.
\end{itemize}

We computed the WJS of the ranked list of CAD genes (respectively, isoforms) and the ranked lists of genes (respectively, isoforms) obtained with NES$^2$RA.

\subsubsection{Significance analysis} 

The scores obtained in different comparisons are not directly comparable between them, since they depend on the length of the lists. In order to obtain values that can be compared directly, we used a permutation approach to estimate the distribution of scores of lists of different length. For each length present, we generated 2000 random lists of genes or isoforms, we computed the WJS and we generated a distribution of the scores. Then, we used this distributions to compute p-values.

Using a significance level of 0.05 (?), we obtained n genes and m isoforms with expansion lists enriched in CAD gene. The list of isoforms was also converted in a list of genes.

\subsubsection{Enrichment analysis}

In order to gain insight on the biological role of the identified genes, we performed a functional enrichment analysis...


\section*{Results}
\addcontentsline{toc}{section}{Results} % Adds this section to the table of contents
Matteo

The list of 149 target genes is reported in the Supplementary Material.

Tables of significant genes / isoforms 

results of the enrichment analysis

\section*{Discussion}
\addcontentsline{toc}{section}{Discussion} % Adds this section to the table of contents
Chiara

gene that are already targets of drugs used for cad

interpretation of the enrichment analysis

discussion of new targets and drugs

\section*{Conclusions}
\addcontentsline{toc}{section}{Conclusions} % Adds this section to the table of contents

necessity of experimental validations

\phantomsection
%----------------------------------------------------------------------------------------
%	REFERENCE LIST
%----------------------------------------------------------------------------------------
\phantomsection
\bibliographystyle{unsrt}
\bibliography{biblio}
\begin{thebibliography}{1}
	
	\bibitem{Polamreddy} Polamreddy P, Gattu N. The drug repurposing landscape from 2012 to 2017: evolution, challenges, and possible solutions. Drug Discov Today. 2018 Dec 1.	

	\bibitem{NES2RA} Asnicar F, Masera L, NES2RA: network expansion by stratified variable subsetting and ranking aggregation. INTERNATIONAL JOURNAL OF HIGH PERFORMANCE COMPUTING APPLICATIONS (2018)
	
	\bibitem{PC-alg}Spirtes P and Glymour C An algorithm for fast recovery
of sparse causal graphs. Social Science Computer, Review, (1991).  

	\bibitem{OpenTargetPlatform}Denise Carvalho-Silva, Open Targets Platform: new developments and updates two years on, Nucleic Acids Research 

	\bibitem{realBoinc}Anderson DP, BOINC: A system for public-resource computing and storage. In: Proceedings of the 5th IEEE/ ACM international workshop on grid computing, GRID ‘04,Washington, DC, US (2004)
	
	\bibitem{WGS}Improved Consistent Sampling, Weighted Minhash and L1 Sketching, Sergey Ioffe, 2010, ICDM
	
	\bibitem{boinc} Asnicar, F. et al. NES2RA. Int. J. High Perform. Comput. Appl. 32, 380–392 (2016).
	
	
	\bibitem{fantom} Noguchi, S. et al. FANTOM5 CAGE profiles of human and mouse samples. Sci Data 4, 170112 (2017).
	
	
	\bibitem{open} Koscielny, G. et al. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 45, D985–D994 (2017).
	
	
\end{thebibliography}

%----------------------------------------------------------------------------------------
%	SUPPLEMENTARY MATERIAL
%----------------------------------------------------------------------------------------

\pagebreak
\onecolumn

\section*{SUPPLEMENTARY MATERIAL}
\renewcommand{\arraystretch}{1.05}

\addcontentsline{toc}{section}{Supplementary Material} % Adds this section to the table of contents


\begin{table}[ht]
	\centering
	\begin{tabular}{llllll}
		\hline
		NDUFA4L2 & JAK1 & CDK16 & SRD5A1 & PRKCD & FLT4 \\ 
		FN1 & KCNJ2 & PIM2 & FGFR4 & ABCA1 & DHODH \\ 
		COL4A1 & FDPS & COL27A1 & TUBA1B & ERBB3 & FNTB \\ 
		COL4A2 & CPT2 & ALOX5 & PIK3CG & NR3C1 & TNFSF12 \\ 
		CTGF & LAMB3 & CD70 & GABBR1 & HGF & TGFB3 \\ 
		F5 & PCSK9 & GSK3B & PDE2A & BIRC2 & KCNH8 \\ 
		SEBOX & FLT1 & SLC29A1 & MMP9 & ERAP1 & EGLN2 \\ 
		TGFB1 & VEGFB & PIM3 & PSCA & OPRL1 & METAP2 \\ 
		F2 & POLD1 & CSF1 & CALM1 & GSTP1 & LAMA2 \\ 
		TLR4 & IL1R1 & CASP7 & PDE8B & PRKACA & NDUFA13 \\ 
		COL5A3 & XPNPEP2 & GAA & DNMT3A & RAMP2 & MAPKAPK5 \\ 
		COL1A1 & NR1H3 & PDE4B & IL6 & HSP90AA1 & CD276 \\ 
		FGA & NFE2L2 & PTGS2 & EPHA2 & INSR & HDAC5 \\ 
		FOLR1 & GBA & NCSTN & PIK3CA & MAP3K9 & DRD2 \\ 
		DHCR24 & PLA2G2A & NDUFS2 & TNFRSF8 & TNFSF13 & NTRK1 \\ 
		GLP2R & ACAT1 & TSG101 & LTB4R & CASP8 & PIK3R3 \\ 
		TNFRSF1A & FDFT1 & PLK1 & VDR & CACNG4 & MALT1 \\ 
		AVPR1A & HLA-DRB1 & NDUFB7 & EEF2 & NFE2 & SLC6A9 \\ 
		FGG & IFNGR2 & BCL2L1 & SCN9A & LTBR & HDAC6 \\ 
		NAMPTL & PLA2G7 & IDH2 & PTH1R & EPHB6 & PTGER4 \\ 
		SRD5A3 & CHEK2 & PSMA4 & P4HB & GART & KCNK10 \\ 
		PDE4D & ATP1B3 & ENG & CFD & KCNH2 & CD44 \\ 
		HAMP & ANPEP & EPHA1 & ARAF & S1PR5 & P4HTM \\ 
		RARA & COL6A3 & TNFRSF10B & SIGMAR1 & TUBB & MET \\ 
		NAMPT & COL11A2 & CDK14 & IFNAR2 & KCNC3 &  \\ 
		\hline
	\end{tabular}
	\smallskip
	\caption{List of tagret genes}
\end{table}


\end{document}